\documentclass{article}
%DIF LATEXDIFF DIFFERENCE FILE
%DIF DEL manuscript/submission.tex   Fri Oct  6 11:44:27 2023
%DIF ADD manuscript/main.tex         Wed Oct 11 15:59:58 2023

\usepackage[letterpaper, margin=1in]{geometry}

\usepackage{lineno}
\linenumbers
\usepackage{titlesec}
%\usepackage[none]{hyphenat} % use only if there is a problem
% Use Unicode characters
\usepackage[utf8]{inputenc}

\usepackage{scicite}
% Clean citations with biblatex
\usepackage[
backend=biber,
style=science
]{biblatex}

\addbibresource{zotero.bib}
%DIF 20a20
\addbibresource{addons.bib} %DIF > 
%DIF -------

% Highlighting
\usepackage{color}
\usepackage{soul}
\usepackage{xurl}

% tables
\usepackage{longtable,booktabs,array}
\usepackage{multirow}
\usepackage{afterpage}

% figures
\usepackage{wrapfig}
\usepackage{lscape}
\usepackage{rotating}
\usepackage{graphicx}
\graphicspath{assets}
\usepackage{float}
\floatstyle{plaintop}
\restylefloat{table}

\usepackage{times}


\topmargin 0.0cm
\oddsidemargin 0.2cm
\textwidth 16cm 
\textheight 21cm
\footskip 1.0cm


%The next command sets up an environment for the abstract to your paper.

\newenvironment{sciabstract}{%
\begin{quote} \bf}
{\end{quote}}


% Include your paper's title here

\title{Gut-resident microorganisms and their genes are associated with cognition and neuroanatomy in children}

% Place the author information here.  Please hand-code the contact
% information and notecalls; do *not* use \footnote commands.  Let the
% author contact information appear immediately below the author names
% as shown.  We would also prefer that you don't change the type-size
% settings shown here.

\author{%
    \parbox{\linewidth}{\centering
        Kevin S. Bonham,$^{1}$
        Guilherme Fahur Bottino,$^{1}$
        Shelley Hoeft McCann,$^{1}$
        Jennifer Beauchemin,$^{2}$
        Elizabeth Weisse,$^{3}$
        Fatoumata Barry,$^{2}$
        Rosa Cano Lorente,$^{2}$
        The RESONANCE Consortium,
        Curtis Huttenhower,$^{4}$
        Muriel Bruchhage,$^{3}$
        Viren D'Sa,$^{2}$
        Sean Deoni,$^{2}$
        Vanja Klepac-Ceraj$^{1\ast}$
    }
\\
\normalsize{$^{1}$Department of Biological Sciences, Wellesley College, Wellesley, MA, USA}\\
\normalsize{$^{2}$Rhode Island Hospital, Providence, RI, USA}\\
\normalsize{$^{3}$Department of Psychology, University of Stavenger, Stavanger, Norway}\\
\normalsize{$^{4}$Department of Biostatistics, Harvard T.H. Chan School of Public Health, Boston, MA, USA}\\
\\
\normalsize{$^\ast$To whom correspondence should be addressed; E-mail:  vklepacc@wellesley.edu.}
}

% Include the date command, but leave its argument blank.

\date{}
%DIF PREAMBLE EXTENSION ADDED BY LATEXDIFF
%DIF UNDERLINE PREAMBLE %DIF PREAMBLE
\RequirePackage[normalem]{ulem} %DIF PREAMBLE
\RequirePackage{color}\definecolor{RED}{rgb}{1,0,0}\definecolor{BLUE}{rgb}{0,0,1} %DIF PREAMBLE
\providecommand{\DIFadd}[1]{{\protect\color{blue}\uwave{#1}}} %DIF PREAMBLE
\providecommand{\DIFdel}[1]{{\protect\color{red}\sout{#1}}}                      %DIF PREAMBLE
%DIF SAFE PREAMBLE %DIF PREAMBLE
\providecommand{\DIFaddbegin}{} %DIF PREAMBLE
\providecommand{\DIFaddend}{} %DIF PREAMBLE
\providecommand{\DIFdelbegin}{} %DIF PREAMBLE
\providecommand{\DIFdelend}{} %DIF PREAMBLE
\providecommand{\DIFmodbegin}{} %DIF PREAMBLE
\providecommand{\DIFmodend}{} %DIF PREAMBLE
%DIF FLOATSAFE PREAMBLE %DIF PREAMBLE
\providecommand{\DIFaddFL}[1]{\DIFadd{#1}} %DIF PREAMBLE
\providecommand{\DIFdelFL}[1]{\DIFdel{#1}} %DIF PREAMBLE
\providecommand{\DIFaddbeginFL}{} %DIF PREAMBLE
\providecommand{\DIFaddendFL}{} %DIF PREAMBLE
\providecommand{\DIFdelbeginFL}{} %DIF PREAMBLE
\providecommand{\DIFdelendFL}{} %DIF PREAMBLE
\newcommand{\DIFscaledelfig}{0.5}
%DIF HIGHLIGHTGRAPHICS PREAMBLE %DIF PREAMBLE
\RequirePackage{settobox} %DIF PREAMBLE
\RequirePackage{letltxmacro} %DIF PREAMBLE
\newsavebox{\DIFdelgraphicsbox} %DIF PREAMBLE
\newlength{\DIFdelgraphicswidth} %DIF PREAMBLE
\newlength{\DIFdelgraphicsheight} %DIF PREAMBLE
% store original definition of \includegraphics %DIF PREAMBLE
\LetLtxMacro{\DIFOincludegraphics}{\includegraphics} %DIF PREAMBLE
\newcommand{\DIFaddincludegraphics}[2][]{{\color{blue}\fbox{\DIFOincludegraphics[#1]{#2}}}} %DIF PREAMBLE
\newcommand{\DIFdelincludegraphics}[2][]{% %DIF PREAMBLE
\sbox{\DIFdelgraphicsbox}{\DIFOincludegraphics[#1]{#2}}% %DIF PREAMBLE
\settoboxwidth{\DIFdelgraphicswidth}{\DIFdelgraphicsbox} %DIF PREAMBLE
\settoboxtotalheight{\DIFdelgraphicsheight}{\DIFdelgraphicsbox} %DIF PREAMBLE
\scalebox{\DIFscaledelfig}{% %DIF PREAMBLE
\parbox[b]{\DIFdelgraphicswidth}{\usebox{\DIFdelgraphicsbox}\\[-\baselineskip] \rule{\DIFdelgraphicswidth}{0em}}\llap{\resizebox{\DIFdelgraphicswidth}{\DIFdelgraphicsheight}{% %DIF PREAMBLE
\setlength{\unitlength}{\DIFdelgraphicswidth}% %DIF PREAMBLE
\begin{picture}(1,1)% %DIF PREAMBLE
\thicklines\linethickness{2pt} %DIF PREAMBLE
{\color[rgb]{1,0,0}\put(0,0){\framebox(1,1){}}}% %DIF PREAMBLE
{\color[rgb]{1,0,0}\put(0,0){\line( 1,1){1}}}% %DIF PREAMBLE
{\color[rgb]{1,0,0}\put(0,1){\line(1,-1){1}}}% %DIF PREAMBLE
\end{picture}% %DIF PREAMBLE
}\hspace*{3pt}}} %DIF PREAMBLE
} %DIF PREAMBLE
\LetLtxMacro{\DIFOaddbegin}{\DIFaddbegin} %DIF PREAMBLE
\LetLtxMacro{\DIFOaddend}{\DIFaddend} %DIF PREAMBLE
\LetLtxMacro{\DIFOdelbegin}{\DIFdelbegin} %DIF PREAMBLE
\LetLtxMacro{\DIFOdelend}{\DIFdelend} %DIF PREAMBLE
\DeclareRobustCommand{\DIFaddbegin}{\DIFOaddbegin \let\includegraphics\DIFaddincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFaddend}{\DIFOaddend \let\includegraphics\DIFOincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFdelbegin}{\DIFOdelbegin \let\includegraphics\DIFdelincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFdelend}{\DIFOaddend \let\includegraphics\DIFOincludegraphics} %DIF PREAMBLE
\LetLtxMacro{\DIFOaddbeginFL}{\DIFaddbeginFL} %DIF PREAMBLE
\LetLtxMacro{\DIFOaddendFL}{\DIFaddendFL} %DIF PREAMBLE
\LetLtxMacro{\DIFOdelbeginFL}{\DIFdelbeginFL} %DIF PREAMBLE
\LetLtxMacro{\DIFOdelendFL}{\DIFdelendFL} %DIF PREAMBLE
\DeclareRobustCommand{\DIFaddbeginFL}{\DIFOaddbeginFL \let\includegraphics\DIFaddincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFaddendFL}{\DIFOaddendFL \let\includegraphics\DIFOincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFdelbeginFL}{\DIFOdelbeginFL \let\includegraphics\DIFdelincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFdelendFL}{\DIFOaddendFL \let\includegraphics\DIFOincludegraphics} %DIF PREAMBLE
%DIF COLORLISTINGS PREAMBLE %DIF PREAMBLE
\RequirePackage{listings} %DIF PREAMBLE
\RequirePackage{color} %DIF PREAMBLE
\lstdefinelanguage{DIFcode}{ %DIF PREAMBLE
%DIF DIFCODE_UNDERLINE %DIF PREAMBLE
  moredelim=[il][\color{red}\sout]{\%DIF\ <\ }, %DIF PREAMBLE
  moredelim=[il][\color{blue}\uwave]{\%DIF\ >\ } %DIF PREAMBLE
} %DIF PREAMBLE
\lstdefinestyle{DIFverbatimstyle}{ %DIF PREAMBLE
	language=DIFcode, %DIF PREAMBLE
	basicstyle=\ttfamily, %DIF PREAMBLE
	columns=fullflexible, %DIF PREAMBLE
	keepspaces=true %DIF PREAMBLE
} %DIF PREAMBLE
\lstnewenvironment{DIFverbatim}{\lstset{style=DIFverbatimstyle}}{} %DIF PREAMBLE
\lstnewenvironment{DIFverbatim*}{\lstset{style=DIFverbatimstyle,showspaces=true}}{} %DIF PREAMBLE
%DIF END PREAMBLE EXTENSION ADDED BY LATEXDIFF

\begin{document}

\baselineskip24pt

% Make the title.
\maketitle 

\begin{abstract}
The gastrointestinal tract, its resident microorganisms, and the central
nervous system are connected by biochemical signaling, also known as the
"microbiome-gut-brain-axis." Both the human brain and the gut microbiome
have critical developmental windows in the first years of life,
raising the possibility that their development is co-occurring and
likely co-dependent. Emerging evidence implicates gut microorganisms and
microbiota composition in cognitive outcomes and neurodevelopmental
disorders (e.g., autism and anxiety), but the influence of gut microbial
metabolism on typical neurodevelopment has not been explored in detail.
We investigated the relationship of the microbiome with the neuroanatomy
and cognitive function of \DIFdelbegin \DIFdel{361 }\DIFdelend \DIFaddbegin \DIFadd{381 }\DIFaddend healthy children, demonstrating that
differences in gut microbial taxa and gene functions are associated with
overall cognitive function and with differences in the size of
multiple brain regions.
Using a combination of multivariate linear and machine learning (ML) models,
we showed that many species,
including \emph{\DIFdelbegin \DIFdel{Gordonibacter pamelae}\DIFdelend \DIFaddbegin \DIFadd{Alistipes obesi}\DIFaddend } and \emph{Blautia wexlerae}, were
\DIFdelbegin \DIFdel{significantly }\DIFdelend associated with higher cognitive function, while some
species such as \emph{Ruminococcus gnavus} were more commonly found in
children with low cognitive scores after controlling for sociodemographic factors.
Microbial genes for enzymes involved in the metabolism of
neuroactive compounds, particularly short-chain fatty acids such as
acetate and propionate, were also associated with cognitive function.
In addition, ML models were able to use microbial taxa to predict the
volume of brain regions, and many taxa that were identified
as important in predicting cognitive function also dominated the
feature importance metric for individual brain regions\DIFaddbegin \DIFadd{,
and for specific subscales of cognitive function}\DIFaddend .
For example, \emph{B. wexlerae}
was the most important species in models predicting the size of the parahippocampal region
in both the left and right hemispheres
\DIFdelbegin \DIFdel{, while several }\DIFdelend \DIFaddbegin \DIFadd{and was among the top predictors of gross motor and expressive language performance.
Several }\DIFaddend species from
the phylum \textit{Bacteroidetes}, including GABA-producing
\emph{B. ovatus}, were important for predicting
the size of the left accumbens area, but not the right. These
findings provide potential biomarkers of neurocognition and brain development
and may lead to the future development of targets for early detection and early
intervention.
\end{abstract}

% Teaser
% This one-sentence summary provides a snapshot of your research for
% non-specialist readers and will be used to promote your article if accepted for publication.
% These should complement rather than repeat the title. 

% The microorganisms that colonize the gut of humans can influence
% health and disease in numerous


\section*{Introduction}

The gut and the brain are intimately linked. Signals from the brain
reach the gut through the autonomic nervous system and the endocrine
system, and the gut can communicate with the brain through the vagus
nerve and through endocrine and immune (cytokine) signaling molecules
\cite{cerdoEarlyNutritionGut2019,pronovostPerinatalInteractionsMicrobiome2019,sharonCentralNervousSystem2016,togniniGutMicrobiotaPotential2017}.
In addition, the products of microbial metabolism generated in the gut can
influence the brain, both indirectly by stimulating the enteric nervous
and immune systems, as well as directly through molecules that enter
circulation and cross the blood-brain barrier. Causal links between the
gut microbiome and neural development, particularly atypical
development, are increasingly being identified
\cite{spichakMiningMicrobesMental2021}.
Both human epidemiology and animal models point to the effects
of gut microbes on the development of autism spectrum disorder
\cite{laueProspectiveAssociationsInfant2020,wanUnderdevelopmentGutMicrobiota2021},
and specific microbial taxa have been associated with depression
\cite{mayneris-perxachsMicrobiotaAlterationsProline2022,valles-colomerNeuroactivePotentialHuman2019}
and Alzheimer's disease
\cite{fungInteractionsMicrobiotaImmune2017,kimProbioticSupplementationImproves2021}.
However, information about this "microbiome-gut-brain
axis" in normal neurocognitive development remains lacking,
particularly early in life.

The first years of life are critical developmental windows for both the
microbiome and the brain
\cite{laueDevelopingMicrobiomeBirth2022}.
Fetal development is believed to occur in a sterile environment, but newborns
are rapidly seeded at birth through contact with the birth canal (if
birthed vaginally), caregivers, food (breast milk or formula),
and other environmental sources
\cite{backhedDynamicsStabilizationHuman2015,bokulichAntibioticsBirthMode2016}.
The early microbiome is characterized by low microbial
diversity, rapid succession and evolution, and \DIFdelbegin \DIFdel{domination }\DIFdelend \DIFaddbegin \DIFadd{is dominated }\DIFaddend by
Actinobacteria, particularly the genus \emph{Bifidobacterium},
Bacteroidetes, especially \emph{Bacteroides}, and Proteobacteria
\cite{koenigSuccessionMicrobialConsortia2011}.
Many of these bacteria have specialized metabolic capabilities
for digesting human \DIFdelbegin \DIFdel{breast }\DIFdelend milk, such as \emph{Bifidobacterium longum}
subsp. \textit{infantis} and \emph{Bacteroides fragilis}
\cite{selaGenomeSequenceBifidobacterium2008}.
Upon the introduction of solid foods, the gut
microbiome undergoes another categorical transformation;
its diversity increases, and most taxa of the infant microbiome
are replaced by taxa more reminiscent of adult microbiomes
\cite{backhedDynamicsStabilizationHuman2015}.
Prior studies have typically focused on either infant microbiomes or
adult microbiomes, since performing statistical analyses across this
transition poses particular challenges. Nevertheless, since this
transition coincides with critical neural developmental windows and
associated neurodevelopmental processes including myelination, neurogenesis, and synaptic pruning,
investigation across this solid-food boundary is important
\cite{tauNormalDevelopmentBrain2010}.

A child's brain undergoes remarkable anatomical, microstructural,
organizational, and functional changes in the first years of life. By
age 5, a child's brain has reached \textgreater85\% of its adult size,
has achieved near-adult levels of myelination, and the pattern of axonal
connections has been established
\cite{silbereisCellularMolecularLandscapes2016}.
Much of this development occurs in discrete windows called
sensitive periods (SPs)
\cite{knudsenSensitivePeriodsDevelopment2004}
during which neural plasticity is particularly high.
Emerging evidence suggests that the timing and duration of SPs
may be driven in part by cues from the developing gut microbiome
\cite{callaghanNestedSensitivePeriods2020,cowanAnnualResearchReview2020}.
As such, understanding the normal spectrum of healthy microbiome
development and how it relates to normal neurocognitive development may
provide opportunities for identifying atypical development earlier and
offer opportunities for intervention.

To begin to address this need, we investigated relationship between
the gut microbiome and
neurocognitive development in a large cohort of healthy and neurotypically-developing
children from infancy through 10 years of age. Gut microbial
communities were assessed using shotgun metagenomic sequencing, enabling
profiling at both the taxonomic and gene-functional level.
Cognitive skills and abilities were measured using age-appropriate
psychometric assessments of cognitive function, i.e.,
the Mullen Scales of Early Learning (MSEL) \cite{mullenMullenScalesEarly1995},
Weschler Preschool and Primary Scale of Intelligence, 4th Edition (WPPSI-4) 
\cite{wechslerWechslerPreschoolPrimary2012}
and Weschler Intelligence Scale for Children, 5th Edition (WISC-V)
\cite{wechslerWechslerIntelligenceScale1949}.
Finally, we also assessed emerging brain structure using magnetic resonance imaging
(MRI). Through a combination of classical statistical analysis and machine
learning, we find that the development of the gut microbiome,
children's cognitive abilities, and brain structure are intimately linked, with
both microbial taxa and gene functions able to predict cognitive
performance and brain structure.

\section*{Results}

\subsubsection*{Cohort overview and summary data}

We investigated the co-development of the brain and microbiome in
\DIFdelbegin \DIFdel{361 }\DIFdelend \DIFaddbegin \DIFadd{381 }\DIFaddend healthy and neurotypically-developing children (\DIFdelbegin \DIFdel{163 }\DIFdelend \DIFaddbegin \DIFadd{172 }\DIFaddend female) between \DIFdelbegin \DIFdel{82 }\DIFdelend \DIFaddbegin \DIFadd{40 }\DIFaddend days and 10 years of age
(Table 1,  Figure 1A, Figure S1)
using a variety of orthogonal microbial and neurocognitive measures.
These included shotgun metagenomic sequencing,
age-appropriate cognitive and behavioral assessments
using full-scale composite scores from the MSEL, WPPSI-4 and WISC-V,
and neuroimaging measures of cortical and sub-cortical morphology. 
Different assessment instruments were used depending on the child's age
due to the unique age-range and psychometric properties measured by each
(i.e., the MSEL for children under 4 years of age, The WPPSI-3 for 3-6 year-olds,
and the WISC-V for children older than 6), but were normalized to a common scale (Figure 1B),
As expected, the greatest differences in microbial taxa were observed across
child age (Figure 1C-D, PCoA axis 1), with older children primarily
stratified into Bacteroidetes-dominant, Firmicutes-dominant, or
high-abundance \emph{Prevotella copri} (Figure S2).
Overall subject-to-subject variation in gut microbial genes
and brain volume profiles was similarly driven largely by subject age (Figure 1E-F).

\begin{sidewaysfigure}[!h]
    \centering
    \DIFaddbegin \small
    \DIFaddend \includegraphics[width=0.9\textwidth]{assets/Figure1.png}
    \caption{
        \label{fig:1} Study design and overview. (\textbf{A}) Stool samples,
        cognitive assessments and neuroimaging were collected from participants
        at different ages throughout the first years of life.
        (\textbf{B}) Cognitive function scores are assessed by
        different instruments, but may be normalized using full-scale composite scores.
        (\textbf{C-D}) Principal coordinates analysis using Bray-Curtis dissimilarity on
        taxonomic profiles demonstrates high beta diversity, with much of the
        first axis of variation explained by increasing age and alpha diversity.
        Differences in gene function profiles (\textbf{E}) and neuroimaging (PCA based on
        the Euclidean distance of brain region volumes) (\textbf{F}) are likewise
        dominated by changes as children age. (\textbf{G}) Permutation analysis of
        variance (PERMANOVA) of taxonomic profiles, functional profiles
        (annotated as UniRef90s) and neuroimaging, against the metadata of
        interest. Variations in taxonomic and functional profiles explain a
        modest but significant percent of the variation in cognitive development
        in children over 18 months of age. (\textbf{H}) Mantel testing of different
        microbial feature matrices, shows overlapping but distinct patterns of
        variation. Dotted lines in (B) and (C) show 6 and 18 months, which are
        used as cut-offs in some following models.
    }

\end{sidewaysfigure}

\begin{table}[!h]
    \centering
    \begin{tabular}{|r|r|r|r|r|r|}
      \hline\hline
      \textbf{Group}              & \textbf{Subgroup}        & \textbf{All} & \textbf{Under 6 months} & \textbf{Over 18 months} & \textbf{Future} \\\hline
      N subjects                  &                          & 381 & 82 & 277 & 104 \\  \hline
    \multirow{3}{*}{Samples}      & 1                        & 214 (56.2\%) &  &  &  \\ \cline{2-6}
                                  & 2                        & 90 (23.6\%) &  &  &  \\ \cline{2-6}
                                  & \textgreater 2           & 77 (20.2\%) &  &  &  \\ \hline
    \multirow{3}{*}{Age (months)} & min                      & 1.3 & 2.89 & 18.03 & 1.3 \\ \cline{2-6}
                                  & max                      & 119.53 & 5.95 & 119.53 & 11.95 \\ \cline{2-6}
                                  & median                   & 24.62 & 3.64 & 41.59 & 6.08 \\ \hline
    \multirow{2}{*}{Sex}          & F                        & 172 (45.1\%) & 42 (51.2\%) & 122 (44.0\%) & 47 (45.2\%) \\   \cline{2-6}
                                  & M                        & 209 (54.9\%) & 40 (48.8\%) & 155 (56.0\%) & 57 (54.8\%) \\  \hline
    \multirow{6}{*}{Race}         & White                    & 265 (69.6\%) & 48 (58.5\%) & 201 (72.6\%) & 62 (59.6\%) \\   \cline{2-6}
                                  & Black                    & 40 (10.5\%) & 15 (18.3\%) & 21 (7.6\%) & 17 (16.3\%) \\ \cline{2-6}
                                  & Indigenous               & 1 (0.3\%) & 0 (0.0\%) & 1 (0.4\%) & 0 (0.0\%) \\ \cline{2-6}
                                  & Asian                    & 8 (2.1\%) & 2 (2.4\%) & 7 (2.5\%) & 3 (2.9\%) \\ \cline{2-6}
                                  & Mixed                    & 54 (14.2\%) & 14 (17.1\%) & 38 (13.7\%) & 16 (15.4\%) \\ \cline{2-6}
                                  & Other                    & 3 (0.8\%) & 1 (1.2\%) & 3 (1.1\%) & 2 (1.9\%) \\ \hline
    \multirow{6}{*}{Maternal Ed.} & Junior high school       & 2 (0.5\%) & 0 (0.0\%) & 2 (0.7\%) & 0 (0.0\%) \\   \cline{2-6}
                                  & Some high school         & 6 (1.6\%) & 3 (3.7\%) & 4 (1.4\%) & 2 (1.9\%) \\ \cline{2-6}
                                  & High school grad         & 39 (10.2\%) & 13 (15.9\%) & 22 (7.9\%) & 17 (16.3\%) \\ \cline{2-6}
                                  & Some college             & 100 (26.2\%) & 28 (34.1\%) & 60 (21.7\%) & 30 (28.8\%) \\ \cline{2-6}
                                  & College grad             & 95 (24.9\%) & 19 (23.2\%) & 75 (27.1\%) & 30 (28.8\%) \\ \cline{2-6}
                                  & Grad/professional school & 126 (33.1\%) & 16 (19.5\%) & 106 (38.3\%) & 23 (22.1\%) \\ \hline\hline
    \end{tabular}
    \caption{\label{tab:demo}Subject demographics of the ECHO RESONANCE cohort.
            "All" refers to the entire cohort.
            "Under 6 months", "Over 18 months", and "Future" refer specifically
            to samples that were used included in the respective subgroup analyses.
    }
\end{table}


As several prior studies have demonstrated associations between specific gut taxa
and atypical neurocognition and neurological disorders
\cite{liuAlteredGutMicrobiota2019,wanUnderdevelopmentGutMicrobiota2021,
      magnussonRelationshipsDietrelatedChanges2015,mayneris-perxachsMicrobiotaAlterationsProline2022,
      needhamGutderivedMetaboliteAlters2022},
we sought to determine if specific taxa or gene functions were associated
with normal cognitive development in children.
To test if variations in gut microbial taxa,
their genes, or their metabolism were associated with neurocognitive
development, we used permutation analysis of variance (PERMANOVA) 
that included the $\beta$ diversity of microbial taxa and gene functions,
neuroimaging-derived volume measures of cortical and subcortical structures,
and general cognitive ability.
Given the large ecological shift in the microbiome that occurs upon the introduction of solid food,
and the relatively wide range of ages when over which infant transition from milk to solid foods,
we separately considered ages that are generally pre-transition (prior to 6 months of age old)
and those that are generally post-transition (over 18 months of age).
However, even with this categorization, we find increasing variation
in gut microbiomes in children over 18 months old (Figure 1G).
We also found that overall variation in microbial species in children over 18 months old
was significantly associated with variation in cognitive function score
(Figure 1G, R\textsuperscript{2} = \DIFdelbegin \DIFdel{0.0124}\DIFdelend \DIFaddbegin \DIFadd{0.0099}\DIFaddend , q \textless{} \DIFdelbegin \DIFdel{0.001}\DIFdelend \DIFaddbegin \DIFadd{0.01}\DIFaddend ),
as was variation in microbial gene functions (R\textsuperscript{2} = \DIFdelbegin \DIFdel{0.0119}\DIFdelend \DIFaddbegin \DIFadd{0.0115}\DIFaddend , q \textless{} 0.001).
Variation in microbial taxa and genes was not significantly associated with cognitive
function in children under 6 months, though this may be due to the low
taxonomic diversity and broad lack of overlap between taxa in infants (Figure 1G).
As expected, age was significantly associated with microbial beta
diversity (taxa R\textsuperscript{2} = \DIFdelbegin \DIFdel{0.0215}\DIFdelend \DIFaddbegin \DIFadd{0.0207}\DIFaddend , and  gene functions
annotated with UniRef90s clusters of 90\% similarity
\cite{suzekUniRefComprehensiveNonredundant2007} -
R\textsuperscript{2} = \DIFdelbegin \DIFdel{0.0245}\DIFdelend \DIFaddbegin \DIFadd{0.0258}\DIFaddend , q \textless{} 0.001), and very strongly
associated with variation in neuroimaging \DIFaddbegin \DIFadd{(MRI) }\DIFaddend profiles (R\textsuperscript{2} = \DIFdelbegin \DIFdel{0.256}\DIFdelend \DIFaddbegin \DIFadd{0.258}\DIFaddend , q
\textless{} 0.001).

Consistent with prior studies, different microbial measurement types
captured overlapping variation, with species profiles and gene function
profiles, both generated from metagenomic sequences, being tightly coupled
(Figure 1H, p \textless{} 0.001). Other functional
groupings (Enzyme commission level4 - ECs
\cite{bairochENZYMEDatabase20002000},
and KEGG orthologues - KOs
\cite{kanehisaKEGGResourceDeciphering2004})
overlapped only slightly with taxonomic profiles in both age
cohorts, despite being derived from UniRef90 labels. In children over 18
months, some variation (\DIFdelbegin \DIFdel{15.9}\DIFdelend \DIFaddbegin \DIFadd{taxa - 20.1}\DIFaddend \%, p \textless{} 0.01\DIFaddbegin \DIFadd{;
gene functions - 15.9\%, p \textless{} 0.05}\DIFaddend ) in neuroimaging
overlapped with microbial measures, though this may be due to the
residual variation due to age in both measures.

\subsubsection*{Microbial species and neuroactive genes are associated with cognitive performance}

To assess whether individual microbial species were associated with
cognitive function, we fit multivariable linear regression models (LMs)
\cite{mallickMultivariableAssociationDiscovery2021}
for the relative abundance of each species that had at least 15\%
prevalence in a given age group (Figure 2A, N\textsubscript{species} = \DIFdelbegin \DIFdel{92 }\DIFdelend \DIFaddbegin \DIFadd{116 }\DIFaddend for 0--120 months, 
N\textsubscript{species} = \DIFdelbegin \DIFdel{46 }\DIFdelend \DIFaddbegin \DIFadd{54 }\DIFaddend for 0--6 months, N\textsubscript{species} = \DIFdelbegin \DIFdel{97 }\DIFdelend \DIFaddbegin \DIFadd{136 }\DIFaddend for 18--120 months).
\DIFdelbegin \DIFdel{Only }\emph{\DIFdel{Blautia wexlerae}} %DIFAUXCMD
\DIFdel{was }\DIFdelend \DIFaddbegin \DIFadd{No species were }\DIFaddend significantly associated (q \DIFdelbegin \DIFdel{value = 0.14, $\beta$ = 0.0015}\DIFdelend \DIFaddbegin \DIFadd{\textless{} 0.20}\DIFaddend ) with
cognitive function in children under 6 months old after adjusting for
age and maternal education\DIFdelbegin \DIFdel{(Figure 2B)}\DIFdelend .
By contrast, in children over 18 months of age,
several mirobial species were significantly enriched
\DIFdelbegin \DIFdel{(q-value
\textless{} 0.20) }\DIFdelend in children with higher cognitive function scores \DIFdelbegin \DIFdel{,
including }\DIFdelend \DIFaddbegin \DIFadd{(Figure 2B),
including }\DIFaddend \emph{\DIFdelbegin \DIFdel{Gordonibacter pamelaeae}\DIFdelend \DIFaddbegin \DIFadd{Alistipes obesi}\DIFaddend },
\emph{Asaccharobacter celatus} \DIFdelbegin \DIFdel{and }\emph{\DIFdel{Adelcreutzia equolifaciens}}
%DIFAUXCMD
\DIFdelend (also known as \textit{Adlercreutzia equolifaciens} subsp. \textit{celatus}
\DIFdelbegin \DIFdel{and }\textit{\DIFdel{Adlercreutzia equolifaciens}} %DIFAUXCMD
\DIFdel{subsp. }\textit{\DIFdel{equolifaciens}} %DIFAUXCMD
\DIFdel{respectively
}\DIFdelend \cite{maruoAdlercreutziaEquolifaciensGen2008}),
and SCFA-producing probiotic
species such as \emph{Eubacterium eligens} and \emph{Faecalibacterium
prausnitzii}
\cite{ghoshMediterraneanDietIntervention2020}
(Figure 2B). \textit{\DIFdelbegin \DIFdel{Ruminococcus gnavus}\DIFdelend \DIFaddbegin \DIFadd{Sutterella wadsworthensis}\DIFaddend } was the only microbial species
that was significantly negatively associated with cognitive function score (q value = \DIFdelbegin \DIFdel{0.18}\DIFdelend \DIFaddbegin \DIFadd{0.165}\DIFaddend ).

\begin{sidewaysfigure}[!htb]
    \centering
    \DIFdelbegin %DIFDELCMD < \includegraphics[width=\textwidth]{assets/Figure2.png}
%DIFDELCMD <     %%%
\DIFdelend \DIFaddbegin \small
    \includegraphics[width=0.9\textwidth]{assets/Figure2.png}
    \DIFaddend \caption{
        Taxa and gene functional groups are associated with cognitive function.
        (\textbf{A}) Volcano plot of multivariable linear model results showing the
        relationship between individual taxa and cognitive function score in
        children over 18 months of age \DIFaddbegin \DIFadd{(left) or across all ages (right)}\DIFaddend ,
        controlling for age, maternal education, and sequencing depth.
        Taxa that were significant after FDR correction (q
        \textless{} 0.2) are colored red. (\textbf{B}) For taxa that were significantly
        associated with cognitive function,
        heatmaps of \DIFdelbegin \DIFdel{prevalence, mean
        relative abundance, and }\DIFdelend correlation with cognitive function in different age groups.
        (\DIFdelbegin \textbf{\DIFdel{C-F}}%DIFAUXCMD
\DIFdelend \DIFaddbegin \textbf{\DIFadd{C}}\DIFaddend ) \DIFaddbegin \DIFadd{Multivariate linear models showing the relationship
        between individual taxa and domain-specific skill assessments
        from the MSEL for children under 6m (left), 18m to 36m (right),
        or over across all ages (right)
        controlling for age, maternal education, and sequencing depth.
        Asterisks in (A) and (C) indicate significance (q \textless{} 0.2)
        after multiple hypothesis correction.
        (}\textbf{\DIFadd{D-E}}\DIFadd{) }\DIFaddend Enrichment plots for \DIFdelbegin \DIFdel{selected neuroactive }\DIFdelend \DIFaddbegin \DIFadd{proprionate degradation I }\DIFaddend gene \DIFdelbegin \DIFdel{sets
        }\DIFdelend \DIFaddbegin \DIFadd{set
        }\DIFaddend used in feature set enrichment analysis (FSEA) \DIFaddbegin \DIFadd{in children under 6m (D)
        or over 18m (E)}\DIFaddend . (\textbf{G}) summary of FSEA
        results \DIFdelbegin \DIFdel{across all samples (left) as well as }\DIFdelend in \DIFdelbegin \DIFdel{the }\DIFdelend \DIFaddbegin \DIFadd{children }\DIFaddend under \DIFdelbegin \DIFdel{6 months
        }\DIFdelend \DIFaddbegin \DIFadd{6m of age }\DIFaddend (\DIFdelbegin \DIFdel{middle}\DIFdelend \DIFaddbegin \DIFadd{left}\DIFaddend )
        \DIFdelbegin \DIFdel{and }\DIFdelend over 18 months (\DIFaddbegin \DIFadd{middle) or across all ages (}\DIFaddend right)\DIFdelbegin \DIFdel{subsets}\DIFdelend , colored based on the
        significance of the association. Markers indicate the individual
        correlation of genes within a gene set,
        and vertical bars \DIFaddbegin \DIFadd{colored bars }\DIFaddend represent the
        median correlation of that gene set.
        \DIFaddbegin \DIFadd{Vertical dotted lines within each panel represent the median T statistic
        for all genes in that age group.
    }\DIFaddend }
    \label{fig:2}
\end{sidewaysfigure}

\DIFaddbegin \DIFadd{Cognitive assessments (MSEL, WPPSI, WISC)
are composites of multiple subscales that assess different cognitive domains.
While the composite scores are broadly analagous accross ages and test modality,
the subscales assessed in each test are different.
To assess whether microbial taxa were associated with
the development of specific cognitive domains,
we fit multivariate linear models for each species present in at least 10\%
of samples for children under 6 months, 18 to 36 months, or all ages
with a concurrent MSEL score (Figure 2C).
Several taxa associated with composite scores (Figure 2B)
were also associated with one or more subscores, including
}\textit{\DIFadd{F. prausnitzii}} \DIFadd{and }\textit{\DIFadd{A. obesi}}\DIFadd{.
Interestingly, while no taxa were significantly associated
with composite scores in children under 6 months old,
two species from the genus }\textit{\DIFadd{Streptococcus}}
\DIFadd{(}\textit{\DIFadd{S. peroris}} \DIFadd{and }\textit{\DIFadd{S. mitis}}\DIFadd{), as well as
}\textit{\DIFadd{Bacteroides fragilis}}\DIFadd{, were positively associated
with expressive language in this age group.
}\textit{\DIFadd{S. mitis}} \DIFadd{was also associated with expressive language
when looking at all ages, while }\textit{\DIFadd{Eisenbergiella massiliensis}}\DIFadd{,
}\textit{\DIFadd{Anaerotruncus colihominis}}\DIFadd{, and an unclassified }\textit{\DIFadd{Blautia}}
\DIFadd{species were negatively associated. %DIF >  Add more here if they match brain?
}


\DIFaddend Given that different microbial species might occupy the same metabolic
niche in different individuals, we hypothesized that microbial genes
grouped by functional activity would be associated with cognition. To
test this, we performed feature set enrichment analysis (FSEA) on groups
of genes with neuroactive potential
\cite{valles-colomerNeuroactivePotentialHuman2019}
and concurrent cognitive ability score (Table 2, Figure \DIFdelbegin \DIFdel{2C-G}\DIFdelend \DIFaddbegin \DIFadd{2D-F}\DIFaddend ) and found that
several metabolic pathways were either significantly enriched or depleted in
children with higher cognitive ability scores.
\DIFdelbegin \DIFdel{This was true both when
considering all ages (0-120 months) together, though the enrichment of most
pathways was more pronounced either in children under 6 months or those over 18
months. }\DIFdelend For example, genes for degrading the 3-carbon SCFA, propionate,
were significantly depleted in children with higher cognitive ability
scores \DIFdelbegin \DIFdel{across all age groups tested }\DIFdelend \DIFaddbegin \DIFadd{in children under 6m }\DIFaddend (Table 2; propionate degradation I,
\DIFdelbegin \DIFdel{under 6 months, }\DIFdelend enrichment score (E.S.) = \DIFdelbegin \DIFdel{-0.542}\DIFdelend \DIFaddbegin \DIFadd{-0.66}\DIFaddend , corrected p-value (q)
= \DIFdelbegin \DIFdel{0.020; over 18 months, }\DIFdelend \DIFaddbegin \DIFadd{0.029), as were genes for synthesizing the 2 carbon SCFA acetate
(acetate synthesis II, }\DIFaddend E.S. = \DIFdelbegin \DIFdel{-0.674}\DIFdelend \DIFaddbegin \DIFadd{-0.491}\DIFaddend , q = \DIFdelbegin \DIFdel{0.041). Interestingly, genes
for propionate synthesis
were also significantly depleted in higher
scoring children over 18 months (}\DIFdelend \DIFaddbegin \DIFadd{0.059; acetate synthesis
III, }\DIFaddend E.S. \DIFdelbegin \DIFdel{-0.676}\DIFdelend \DIFaddbegin \DIFadd{= -0.409}\DIFaddend , q = \DIFdelbegin \DIFdel{0.023), as were genes
for synthesizing the 2 carbon SCFA acetate in children under 6 months
old }\DIFdelend \DIFaddbegin \DIFadd{0.133).
Interstingly, in children over 18m,
these metabolic associations were reversed, with both acetate synthesis
and propionate degredation being enriched in children
with higher cognitive scores
}\DIFaddend (acetate synthesis I, E.S. = \DIFdelbegin \DIFdel{-0.194}\DIFdelend \DIFaddbegin \DIFadd{0.192}\DIFaddend , q = \DIFdelbegin \DIFdel{0.153}\DIFdelend \DIFaddbegin \DIFadd{0.145}\DIFaddend ;
acetate synthesis II, E.S. = \DIFdelbegin \DIFdel{-0.342}\DIFdelend \DIFaddbegin \DIFadd{0.248}\DIFaddend , q = \DIFdelbegin \DIFdel{0.020;
acetate synthesis III}\DIFdelend \DIFaddbegin \DIFadd{0.15;
propionate degredation I}\DIFaddend , E.S. = \DIFdelbegin \DIFdel{-0.31}\DIFdelend \DIFaddbegin \DIFadd{0.154}\DIFaddend , q = \DIFdelbegin \DIFdel{0.052).
In children over 18 months old,
synthesis of menaquinone (vitamin K)
was also negatively associated with cognitive function score (menaquinone synthesis I}\DIFdelend \DIFaddbegin \DIFadd{0.178)}\DIFaddend ,
\DIFaddbegin \DIFadd{as was synthesis of the 4-carbon SCFA butyrate
(butyrate synthesis II, }\DIFaddend E.S. = \DIFdelbegin \DIFdel{-0.170}\DIFdelend \DIFaddbegin \DIFadd{0.414}\DIFaddend , q = \DIFdelbegin \DIFdel{0.0183), while
genes for the synthesis }\DIFdelend \DIFaddbegin \DIFadd{0.178).
Synthesis }\DIFaddend of the amino \DIFdelbegin \DIFdel{acids
glutamate and tryptophan were }\DIFdelend \DIFaddbegin \DIFadd{acid
glutamate was also }\DIFaddend significantly enriched in children with
higher cognitive function scores \DIFaddbegin \DIFadd{in this age group }\DIFaddend (glutamate synthesis \DIFdelbegin \DIFdel{I}\DIFdelend \DIFaddbegin \DIFadd{II}\DIFaddend , E.S. = \DIFdelbegin \DIFdel{0.242}\DIFdelend \DIFaddbegin \DIFadd{0.355}\DIFaddend , 
q = \DIFdelbegin \DIFdel{0.047; tryptophan synthesis, E.S. = 0.119, q = 0.041}\DIFdelend \DIFaddbegin \DIFadd{0.023}\DIFaddend ). Taken together, these results suggest that
microbial metabolic activity, particularly the metabolism (synthesis and
degradation) of neuroactive compounds may have effects on cognitive
development.


\begin{table}[!h]
    \begin{center}
    \begin{tabular}{|r|r|r|r|r|r|r|}
        \hline
        \textbf{Age group}& \multicolumn{2}{c|}{\textbf{0 to 6m}} & \multicolumn{2}{c|}{\textbf{18 to 120m}} \\\hline
        geneset & E.S. & q value & E.S. & q value \\\hline
        Acetate syn. I & -0.165 & 0.284 & 0.192 & \color{red}{\textbf{0.145}} \\
        Acetate syn. II & -0.491 & \color{blue}{\textbf{0.059}} & 0.248 & \color{red}{\textbf{0.15}} \\
        Acetate syn. III & -0.409 & \color{blue}{\textbf{0.133}} & 0.215 & 0.209 \\
        Butyrate syn. II & -0.357 & 0.284 & 0.414 & \color{red}{\textbf{0.178}} \\
        Glutamate syn. II & 0.343 & 0.284 & 0.355 & \color{red}{\textbf{0.023}} \\
        Propionate deg. I & -0.659 & \color{blue}{\textbf{0.029}} & 0.154 & \color{red}{\textbf{0.178}} \\\hline
    \end{tabular}
    \caption{\label{tab:fsea}Feature set enrichment analysis on neuroactive microbial gene sets.
    Red and blue text indicates significant (q value \textless 0.2) genesets with a positive or negative
    pseudomedian, respectively.}
    \end{center}
\end{table}


\subsubsection*{Gut microbial taxonomic and functional profiles predict \DIFaddbegin \DIFadd{concurrently measured }\DIFaddend cognitive function}

FSEA relies on understanding functional relationships between individual
genes. However, because the relationships between individual taxa are
still largely unknown, we turned to Random Forest (RF) models, a family of 
unsupervised non-parametric machine learning (ML) method that enables
the identification of underlying patterns in large numbers of individual features
(here, microbial species).
Previous studies have reported
successful use of RFs for processing highly-dimensional and sparse data
from the domain of genomics
\cite{amaratungaEnrichedRandomForests2008,brieucPracticalIntroductionRandom2018,chenRandomForestsGenomic2012,franzosaGutMicrobiomeStructure2019,stephanRandomForestApproach2015},
along with other works where it was used
to predict cognitive conditions related to Alzheimer's disease in
different scenarios \cite{ardekaniPredictionIncipientAlzheimer2017,velazquezRandomForestModel2021}.
Given that gut microbial profiles, as well as neurocognitive
development, may partially reflect socioeconomic and demographic
factors, we assessed the performance of RF regressors where maternal
education (here used as a proxy of socioeconomic status (SES)),
sex, and age were included as possible predictors, either alone
or in combination with microbial taxonomic profiles (Table 3).

\begin{sidewaysfigure}[!htb]
    \centering
    \DIFdelbegin %DIFDELCMD < \includegraphics[width=\textwidth]{assets/Figure3.png}
%DIFDELCMD <     %%%
\DIFdelend \DIFaddbegin \small
    \includegraphics[width=0.9\textwidth]{assets/Figure3.png}
    \DIFaddend \caption{
        Random forest models predict \DIFdelbegin \DIFdel{concurrently }\DIFdelend \DIFaddbegin \DIFadd{both concurrent and future }\DIFaddend measured cognitive function.
        Comparison of RF predictor importance versus linear models for children
        \DIFdelbegin \DIFdel{between birth and 6 }\DIFdelend \DIFaddbegin \DIFadd{over 18 }\DIFaddend months \DIFdelbegin \DIFdel{old }\DIFdelend (\textbf{A})\DIFdelbegin \DIFdel{, and for those older than 18 months
        (}\textbf{\DIFdel{B}}%DIFAUXCMD
\DIFdel{)}\DIFdelend .
        Colors represent whether the species belong to the group of
        top-important features that account for 60\% of the cumulative
        importance on the RF model, if that species was significant (q
        \textless{} 0.2) in linear models, both, or neither.
        \DIFaddbegin \DIFadd{Comparison of RF predictor importance for microbial features
        between concurrent and future cognitive function measure
        (}\textbf{\DIFadd{B}}\DIFadd{).
        Colors represent whether the species belong to the group of
        top-important features that account for 60\% of the cumulative
        importance on the RF model only for concurrent prediction,
        only for the future prediction, or for both models.
        Features absent of one model are shown on the origin-aligned sideplots.
        Distribution of ages for each microbiome sampling - cognitive assesment dyad (}\textbf{\DIFadd{C}}\DIFadd{).
        Blue violin plot shows distribution of sampling age,
        while red violin plot shows distribution of asessment age.
        Black lines connect ages for each individual subject.      
        }\DIFaddend Ranked feature importance for taxa in RF models for
        \DIFdelbegin \DIFdel{children between birth and 6 months
        old }\DIFdelend \DIFaddbegin \DIFadd{both concurrent }\DIFaddend (\DIFdelbegin \textbf{\DIFdel{C}}%DIFAUXCMD
\DIFdelend \DIFaddbegin \textbf{\DIFadd{D}}\DIFaddend ) \DIFdelbegin \DIFdel{, }\DIFdelend and \DIFdelbegin \DIFdel{for those older than 18 months }\DIFdelend \DIFaddbegin \DIFadd{future }\DIFaddend (\DIFdelbegin \textbf{\DIFdel{D}}%DIFAUXCMD
\DIFdelend \DIFaddbegin \textbf{\DIFadd{E}}\DIFaddend )
        \DIFaddbegin \DIFadd{cognitive function measurement prediction}\DIFaddend .
        \DIFdelbegin \DIFdel{Taxa }\DIFdelend \DIFaddbegin \DIFadd{Red dots indicate how relative importances change (among taxa)
        when demographics (sex, SES) are arred as covariables.
        Abundances and prevalences of taxa }\DIFaddend that are important
        for RF models both for \DIFdelbegin \DIFdel{children under 6 months and for those over 18
        months }\DIFdelend \DIFaddbegin \DIFadd{concurrent }\DIFaddend (\DIFdelbegin \textbf{\DIFdel{E}}%DIFAUXCMD
\DIFdel{), only for children under 6 months (}\DIFdelend \textbf{F}) \DIFdelbegin \DIFdel{, }\DIFdelend and \DIFdelbegin \DIFdel{only for children
        over 18 months }\DIFdelend \DIFaddbegin \DIFadd{future }\DIFaddend (\textbf{G}) \DIFaddbegin \DIFadd{assesment prediction.
        }\DIFaddend }
    \label{fig:3}
\end{sidewaysfigure}



\begin{table}[!h]
    \begin{center}
    \begin{tabular}{|r|r|r|r|r|}
      \hline\hline
      \textbf{Subject Ages (months)} & \textbf{Microbial feature} & \textbf{Demo.} &  \textbf{Test set RMSE (± C.I.)} & \textbf{Test set correlation (± C.I.)} \\\hline
      \multirow{5}{*}{0 to 6} & -    & + & \DIFdelbeginFL \DIFdelFL{13.01 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{13.37 }\DIFaddendFL ± 0.05 & \DIFdelbeginFL \DIFdelFL{-0.14 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{-0.16 }\DIFaddendFL ± 0.01 \\ \cline{2-5}
            & \multirow{2}{*}{taxa}  & - & \DIFdelbeginFL \DIFdelFL{12.66 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{12.99 }\DIFaddendFL ± \DIFdelbeginFL \DIFdelFL{0.05 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{0.04 }\DIFaddendFL & \DIFdelbeginFL \DIFdelFL{-0.1 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{-0.11 }\DIFaddendFL ± 0.01 \\ \cline{3-5}
            &                        & + & \DIFdelbeginFL \DIFdelFL{12.7 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{13.01 }\DIFaddendFL ± \DIFdelbeginFL \DIFdelFL{0.05 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{0.04 }\DIFaddendFL & \DIFdelbeginFL \DIFdelFL{-0.12 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{-0.13 }\DIFaddendFL ± 0.01 \\ \cline{2-5}
            & \multirow{2}{*}{genes} & - & \DIFdelbeginFL \DIFdelFL{12.56 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{12.95 }\DIFaddendFL ± \DIFdelbeginFL \DIFdelFL{0.05 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{0.04 }\DIFaddendFL & \DIFdelbeginFL \DIFdelFL{-0.01 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{-0.05 }\DIFaddendFL ± 0.01 \\ \cline{3-5}
            &                        & + & \DIFdelbeginFL \DIFdelFL{12.56 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{12.95 }\DIFaddendFL ± \DIFdelbeginFL \DIFdelFL{0.05 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{0.04 }\DIFaddendFL & \DIFdelbeginFL \DIFdelFL{-0.02 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{-0.05 }\DIFaddendFL ± 0.01 \\ \cline{1-5}
      \multirow{5}{*}{18 to 120} & - & + & \DIFdelbeginFL \DIFdelFL{16.27 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{17.06 }\DIFaddendFL ± 0.04 & \DIFdelbeginFL \DIFdelFL{0.506 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{0.511 }\DIFaddendFL ± 0.003 \\ \cline{2-5}
            & \multirow{2}{*}{taxa}  & - & \DIFdelbeginFL \DIFdelFL{17.66 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{18.71 }\DIFaddendFL ± 0.04 & \DIFdelbeginFL \DIFdelFL{0.363 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{0.347 }\DIFaddendFL ± 0.003 \\ \cline{3-5}
            &                        & + & \DIFdelbeginFL \DIFdelFL{17.29 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{18.25 }\DIFaddendFL ± 0.04 & 0.429 ± 0.003 \\ \cline{2-5}
            & \multirow{2}{*}{genes} & - & \DIFdelbeginFL \DIFdelFL{18.01 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{18.96 }\DIFaddendFL ± 0.04 & \DIFdelbeginFL \DIFdelFL{0.303 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{0.299 }\DIFaddendFL ± 0.003 \\ \cline{3-5}
            &                        & + & \DIFdelbeginFL \DIFdelFL{17.86 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{18.74 }\DIFaddendFL ± 0.04 & \DIFdelbeginFL \DIFdelFL{0.333 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{0.341 }\DIFaddendFL ± \DIFdelbeginFL \DIFdelFL{0.004 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{0.003 }\DIFaddendFL \\ \hline\hline
    \end{tabular}
    \caption{\label{tab:rfbench}Benchmark metrics for the cognitive assessment score
    prediction models. Confidence intervals (C.I.) are calculated from the
    distribution of metrics from repeated CV at a confidence level of 95.
    Root mean square error (RMSE) was used to estimate how well the random forest model was able to predict the outcome.
    Demo. column indicates whether demographic factors such as maternal education, sex and age were included as predictors.}
    \end{center}
\end{table}

As with LMs, RF models for children under 6 months old
performed poorly (mean test-set correlation \DIFdelbegin \DIFdel{-0.12}\DIFdelend \DIFaddbegin \DIFadd{-0.13}\DIFaddend , mean RMSE \DIFdelbegin \DIFdel{12.70}\DIFdelend \DIFaddbegin \DIFadd{13.01}\DIFaddend ),
but RFs were consistently able to learn the relationship between taxa
and cognitive function scores in children over 18 months of age (mean
test-set correlation 0.429, mean RMSE \DIFdelbegin \DIFdel{17.29).
For both age groups,
species }\DIFdelend \DIFaddbegin \DIFadd{18.25).
Species }\DIFaddend that were important in RF overlapped with those that were
significant when testing the relationship with LMs (Figure
3A-B, Table S1A-B); \DIFdelbegin \DIFdel{except for }\emph{\DIFdel{Eubacterium ramulus}}%DIFAUXCMD
\DIFdel{, all }\DIFdelend \DIFaddbegin \DIFadd{Both analysis identified 
}\textit{\DIFadd{F. prausnitzii}}\DIFadd{, %DIF >  F. prausz was mentioned before, ~ line 395
}\textit{\DIFadd{E. eligens}}\DIFadd{, %DIF >  E. eligens was mentioned before, ~ line 395
}\textit{\DIFadd{Parasutterella excrementihominis}}\DIFadd{, and %DIF >  Not mentioned before
}\textit{\DIFadd{A. obesi}} \DIFadd{as important %DIF >  A. obesi was mentioned before, ~ lines 120, 390
(q \textless 0.2 for LMs, cumulative importance \textless 60\% for RFs).
Other }\DIFaddend taxa that were significant \DIFdelbegin \DIFdel{using LMs belonged to a group
responsible for 60\% of the total importancein RF models }\DIFdelend \DIFaddbegin \DIFadd{in LMs were not identified by RF models
including }\textit{\DIFadd{Eggerthella}} \DIFadd{sp. CAG 298,
and }\textit{\DIFadd{B. intestinihominis}}\DIFadd{. }\textit{\DIFadd{E. ramulus}}
\DIFadd{was just outside our cutoff (64\% cumulative importance). %DIF >  Flag - 64%?
RF models also identified an additional 48 species
that were important for discriminating cognitive function scores
}\DIFaddend (Table 3, Figure 3A). \DIFdelbegin \DIFdel{However, }\DIFdelend \DIFaddbegin \DIFadd{Subject age was consistently ranked highly in
feature importance, which could indicate that decision branches based on
microbial taxa have increased purity when considering the subject's age
or that age itself is a useful predictor.
}

\subsubsection*{\DIFadd{Gut microbial profiles in the first year of life predict future cogntitive function}}

\DIFadd{While LMs did not identify any taxa in }\DIFaddend the \DIFdelbegin \DIFdel{rank-ordering of LM significance and RF importance
had substantive differences.
For example, in children under 6 months,
only }\emph{\DIFdel{B. wexlerae}} %DIFAUXCMD
\DIFdel{was significantly associated with cognitive function in LMs,
but
was ranked 20th in importance for RF models. }\DIFdelend \DIFaddbegin \DIFadd{0-6 month range
that were associated with concurrently measured cognitive function,
we reasoned that developmental trajectories that begin
}\textit{\DIFadd{in utero}} \DIFadd{may not be immediately altered by
microbial exposures, such that microbial influences early in life
may take time to manifest. To test this hypothesis,
we asked whether RF models trained on taxonomic profiles
in the first year of life
could predict }\textit{\DIFadd{future}} \DIFadd{cognitive performance measured (Figure 3B, D).
Models were consistently able to learn microbial species that are associated
with cognitive performance at a future visit
(mean time between visits: 21.5 months, S.D. = 9.9 months; Figure 3C,
mean test set correlation: 0.27).
}\DIFaddend 

\DIFdelbegin \DIFdel{Many taxa highly ranked in importance in RF models in both age
groups, including several that were identified in LMs for children over
}\DIFdelend \DIFaddbegin \begin{table}[!h]
    \begin{center}
    \begin{tabular}{|r|r|r|r|r}
        \hline
        \textbf{\DIFaddFL{Microbial feature}} & \textbf{\DIFaddFL{Demo.}} &  \textbf{\DIFaddFL{Test set Root-mean-square-error (± C.I.)}} & \textbf{\DIFaddFL{Test set correlation (± C.I.)}} \\\hline
        \DIFaddFL{- }& \DIFaddFL{+ }& \DIFaddFL{20.25 ± 0.07 }& \DIFaddFL{0.43 ± 0.0 }\\ \cline{1-4}
        \multirow{2}{*}{taxa} & \DIFaddFL{- }& \DIFaddFL{22.32 ± 0.08 }& \DIFaddFL{0.07 ± 0.0 }\\ \cline{2-4}
            & \DIFaddFL{+ }& \DIFaddFL{21.53 ± 0.08 }& \DIFaddFL{0.27 ± 0.0 }\\ \cline{1-4}
        \multirow{2}{*}{genes} & \DIFaddFL{- }& \DIFaddFL{22.21 ± 0.08 }& \DIFaddFL{0.12 ± 0.0 }\\ \cline{2-4}
            & \DIFaddFL{+ }& \DIFaddFL{22.08 ± 0.08 }& \DIFaddFL{0.15 ± 0.0 }\\\hline\hline
    \end{tabular}
    \caption{\label{tab:rfbench2}\DIFaddFL{Benchmark metrics for the cognitive assessment score
    future prediction models. Confidence intervals (C.I.) are calculated from the
    distribution of metrics from repeated CV at a confidence level of 95.
    Root mean square error (RMSE) was used to estimate how well the random forest model was able to predict the outcome.
    Demo. column indicates whether demographic factors such as maternal education, sex and age were included as predictors.}}
    \end{center}
\end{table}

\DIFadd{Nearly all important taxa in RF models that predicted cognitive function
in children older than }\DIFaddend 18 \DIFdelbegin \DIFdel{months old, including }\emph{\DIFdel{R. gnavus}} %DIFAUXCMD
\DIFdel{(0--6 months, rank = 13;
18--120 months, rank = 7)and }\emph{\DIFdel{G. pamelaeae}} %DIFAUXCMD
\DIFdel{(0--6 months, rank =
16; 18--120 months}\DIFdelend \DIFaddbegin \DIFadd{months were also important in
models that predicted future cognitive function, if they were present (Figure 3B, Table S5).
At the same time, many additional species were important for future prediction models
that were not important in the models measuring concurrent cognitive function.
In particular, feature importance for future prediction models
were dominated by two species of }\textit{\DIFadd{Klebsiella}}\DIFaddend ,
\DIFdelbegin \DIFdel{rank = 20), while others such as }\emph{\DIFdel{Allistipes
finegoldii}} %DIFAUXCMD
\DIFdel{were age-group specific.
A number of taxa that were important }\DIFdelend \DIFaddbegin \textit{\DIFadd{K. pneumoniae}} \DIFadd{and }\textit{\DIFadd{K. varicola}} \DIFadd{(Figure 3D, E).
Many of the other species that ranked highest }\DIFaddend in \DIFdelbegin \DIFdel{RF models were not statistically significant when using LMs after
multiple hypothesis correction. However, these taxa often had small nominal
p-values.
For example, }\emph{\DIFdel{Erysipelatoclostridium ramosum}} %DIFAUXCMD
\DIFdel{was the most important feature in RF models for children under 6 months old and
had an LM p-value of 0.04.
Subject age was consistently ranked highly in
}\DIFdelend feature importance
\DIFaddbegin \DIFadd{were also oportunistic pathogens, or previously reported
to be linked with disease (eg. }\textit{\DIFadd{R. gnavus}}\DIFaddend , \DIFdelbegin \DIFdel{which could indicate that decision branches based on
microbial taxa
have increased purity when considering the  
subject's age
or that age itself is a useful predictor.
}\DIFdelend \DIFaddbegin \textit{\DIFadd{E. coli}}\DIFadd{).
}\DIFaddend 

\DIFaddbegin \subsubsection*{\DIFadd{Gut microbial taxa are associated with specific cognitive functions}}

\DIFadd{Given that the development of different domains of cognitive function
(e.g., gross motor development vs visual perception) may develop at different times
and be affected by different molecular signals,
we sought to determine whether the associations of microbial taxa
with cognitive function are general across all cognitive subdomains.
To accomplish this, we assessed individual subscales of the  
MSEL (birth to 36 months - }\textit{\DIFadd{see Methods}}\DIFadd{). 
Taxonomic profiles coupled with demographics exhibited predictive potential towards three
of five probed subscales (Figure 4A, Table 5), 
namely expressive language ( test-set RMSE 12.05, test-set correlation 0.368), 
gross motor (test-set RMSE 11.07, test-set correlation 0.137) and 
visual reception (test-set RMSE 15.2, test-set correlation 0.333).
Between the subscales, both the predictability (measured by the test-set figures of merit)
and the relative importances of several taxa (Figure 4B) is variable.
Although certain species such as }\textit{\DIFadd{B. longum}}\DIFadd{, }\textit{\DIFadd{F. prausznitzii}} 
\DIFadd{have similar relative importance across all subscales, others are substantially different (Tables S2-S4).
Notably, }\textit{\DIFadd{Bifidobacterium pseudocatenulatum}}\DIFadd{, }\textit{\DIFadd{B. wexlerae}} \DIFadd{and }\textit{\DIFadd{E. eligens}} \DIFadd{are
substantially more important for the prediction of expressive language, while
}\textit{\DIFadd{R. faecis}}\DIFadd{, }\textit{\DIFadd{Streptococcus salivarius}} \DIFadd{and }\textit{\DIFadd{Fusicatenibacter saccharivorans}} \DIFadd{are
distinctly predictive of gross motor, and
}\textit{\DIFadd{Clostridium innocuum}}\DIFadd{, }\textit{\DIFadd{B. vulgatus}} 
\DIFadd{stand out on models targeting visual reception (Supplenmentary Tables S2-4).
}

\begin{table}[!h]
    \begin{center}
        \begin{tabular}{|r|r|r|r|r|}
        \hline
        \textbf{\DIFaddFL{Target Mullen scale}} & \textbf{\DIFaddFL{Microbial feature}} & \textbf{\DIFaddFL{Demo.}} & \textbf{\DIFaddFL{Test set RMSE (± C.I.)}} & \textbf{\DIFaddFL{Test set correlation (± C.I.)}} \\\hline
        \multirow{5}{*}{Expressive Language} & \DIFaddFL{- }& \DIFaddFL{+ }& \DIFaddFL{11.17 ± 0.04 }& \DIFaddFL{0.48 ± 0.0 }\\ \cline{2-5}
            & \multirow{2}{*}{taxa}  & \DIFaddFL{- }& \DIFaddFL{12.37 ± 0.04 }& \DIFaddFL{0.25 ± 0.0 }\\ \cline{3-5}
            &                        & \DIFaddFL{+ }& \DIFaddFL{12.05 ± 0.04 }& \DIFaddFL{0.37 ± 0.0 }\\ \cline{2-5}
            & \multirow{2}{*}{genes} & \DIFaddFL{- }& \DIFaddFL{12.59 ± 0.04 }& \DIFaddFL{0.15 ± 0.0 }\\ \cline{3-5}
            &                        & \DIFaddFL{+ }& \DIFaddFL{12.46 ± 0.04 }& \DIFaddFL{0.21 ± 0.0 }\\ \cline{1-5}
        \multirow{5}{*}{Gross Motor} & \DIFaddFL{- }& \DIFaddFL{+ }& \DIFaddFL{10.53 ± 0.04 }& \DIFaddFL{0.346 ± 0.006 }\\ \cline{2-5}
            & \multirow{2}{*}{taxa}  & \DIFaddFL{- }& \DIFaddFL{11.08 ± 0.04 }& \DIFaddFL{0.134 ± 0.006 }\\ \cline{3-5}
            &                        & \DIFaddFL{+ }& \DIFaddFL{11.07 ± 0.04 }& \DIFaddFL{0.137 ± 0.006 }\\ \cline{2-5}
            & \multirow{2}{*}{genes} & \DIFaddFL{- }& \DIFaddFL{10.98 ± 0.04 }& \DIFaddFL{0.197 ± 0.005 }\\ \cline{3-5}
            &                        & \DIFaddFL{+ }& \DIFaddFL{10.98 ± 0.04 }& \DIFaddFL{0.197 ± 0.005 }\\ \hline
        \multirow{5}{*}{Visual reception} & \DIFaddFL{- }& \DIFaddFL{+ }& \DIFaddFL{14.13 ± 0.04 }& \DIFaddFL{0.462 ± 0.003 }\\ \cline{2-5}
            & \multirow{2}{*}{taxa}  & \DIFaddFL{- }& \DIFaddFL{15.49 ± 0.04 }& \DIFaddFL{0.245 ± 0.004 }\\ \cline{3-5}
            &                        & \DIFaddFL{+ }& \DIFaddFL{15.2 ± 0.04 }& \DIFaddFL{0.333 ± 0.004 }\\ \cline{2-5}
            & \multirow{2}{*}{genes} & \DIFaddFL{- }& \DIFaddFL{15.97 ± 0.04 }& \DIFaddFL{0.067 ± 0.004 }\\ \cline{3-5}
            &                        & \DIFaddFL{+ }& \DIFaddFL{15.84 ± 0.04 }& \DIFaddFL{0.118 ± 0.005 }\\ \hline\hline
        \end{tabular}
        \caption{\label{tab:subscalebenchmark}\DIFaddFL{Benchmark metrics for the Mullen subscales prediction models.
        Confidence intervals (C.I.) are calculated from the
        distribution of metrics from repeated CV at a confidence level of 95.
        Root mean square error (RMSE) was used to estimate how well the random forest model was able to predict the outcome.
        Demo. column indicates whether demographic factors such as maternal education, sex and age were included as predictors.
        Age range, in all cases, was 18-120 months.}}
    \end{center}
\end{table}

\DIFaddend \subsubsection*{Gut microbial taxonomic profiles predict brain structure differences}

\DIFdelbegin \DIFdel{If }\DIFdelend \DIFaddbegin \DIFadd{Given that individual microbes could be associated with particular domains
of cognitive function and not others, if }\DIFaddend there are causal effects of microbial metabolism on cognitive
function, they might be reflected in changes in neuroanatomy. We again
used RF modeling to associate gut taxonomic
profiles with individual brain regions identified in MRI scans
(children over 18 months old, N = \DIFdelbegin \DIFdel{121}\DIFdelend \DIFaddbegin \DIFadd{124}\DIFaddend ).
Some brain regions were more readily
predicted by RF models trained on microbial taxa (Table \DIFdelbegin \DIFdel{4, Table S2}\DIFdelend \DIFaddbegin \DIFadd{5, Table S6}\DIFaddend ), in particular those that were highly correlated with age.
These included the L/R lingual gyrus (mean RF correlations, Left =
\DIFdelbegin \DIFdel{0.421}\DIFdelend \DIFaddbegin \DIFadd{0.444}\DIFaddend , Right = \DIFdelbegin \DIFdel{0.434}\DIFdelend \DIFaddbegin \DIFadd{0.441}\DIFaddend ; importance for age, Left = \DIFdelbegin \DIFdel{7.5}\DIFdelend \DIFaddbegin \DIFadd{6.8}\DIFaddend \%, Right =
\DIFdelbegin \DIFdel{8.3}\DIFdelend \DIFaddbegin \DIFadd{7.7}\DIFaddend \%) and the L/R pericalcarine cortex (mean RF correlations, Left =
\DIFdelbegin \DIFdel{0.200}\DIFdelend \DIFaddbegin \DIFadd{0.267}\DIFaddend , Right = \DIFdelbegin \DIFdel{0.273}\DIFdelend \DIFaddbegin \DIFadd{0.315}\DIFaddend ; importance for age, Left = 3.6\%, Right = \DIFdelbegin \DIFdel{6.3}\DIFdelend \DIFaddbegin \DIFadd{6.2}\DIFaddend \%).
In many cases, however, age was not an important variable in
high-performing models, such as that for the left accumbens area (mean
RF correlations = \DIFdelbegin \DIFdel{0.288}\DIFdelend \DIFaddbegin \DIFadd{0.275}\DIFaddend ; relative age importances = \DIFdelbegin \DIFdel{1.2}\DIFdelend \DIFaddbegin \DIFadd{1.1}\DIFaddend \%).

\begin{table}[!h]
    \begin{center}
    \begin{tabular}{|r|r|r|}
      \hline\hline
      \textbf{Statistic} & \textbf{MAPE} & \textbf{R} \\\hline
      Mean & 0.073 & \DIFdelbeginFL \DIFdelFL{0.137 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{0.146 }\DIFaddendFL \\
      Standard deviation & \DIFdelbeginFL \DIFdelFL{0.028 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{0.029 }\DIFaddendFL & \DIFdelbeginFL \DIFdelFL{0.137 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{0.131 }\DIFaddendFL \\
      Maximum & \DIFdelbeginFL \DIFdelFL{0.206 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{0.205 }\DIFaddendFL & \DIFdelbeginFL \DIFdelFL{0.605 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{0.598 }\DIFaddendFL \\
      75th percentile & \DIFdelbeginFL \DIFdelFL{0.082 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{0.081 }\DIFaddendFL & \DIFdelbeginFL \DIFdelFL{0.195 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{0.212 }\DIFaddendFL \\
      50th percentile & \DIFdelbeginFL \DIFdelFL{0.067 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{0.066 }\DIFaddendFL & \DIFdelbeginFL \DIFdelFL{0.134 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{0.129 }\DIFaddendFL \\
      25th percentile & 0.057 & \DIFdelbeginFL \DIFdelFL{0.062 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{0.073 }\DIFaddendFL \\
      Minimum & \DIFdelbeginFL \DIFdelFL{0.035 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{0.036 }\DIFaddendFL & \DIFdelbeginFL \DIFdelFL{-0.147 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{-0.133 }\DIFaddendFL \\\hline
    \DIFdelbeginFL %DIFDELCMD < \hline
%DIFDELCMD <     %%%
\DIFdelendFL \end{tabular}
    \caption{\label{tab:neuro}Summary statistics for the neuroanatomy prediction benchmarks.
    Mean absolute proportional error (MAPE) and correlation coeficient (R)
    of test-set preditions are reported accross models for all brain regions.
    Complete list of values can be found on Table \DIFdelbeginFL \DIFdelFL{S2}\DIFdelendFL \DIFaddbeginFL \DIFaddFL{S6}\DIFaddendFL .}
    \end{center}
\end{table}

While models for many regions were highly concordant across the
left and right hemispheres in terms of both model performance and
microbial feature importance, other regions had substantial
bilateral differences. For example, the left accumbens area
has one of the highest test-set correlations of our
brain region models (R = 0.288), as compared to the right which
had a \DIFdelbegin \DIFdel{negative mean }\DIFdelend \DIFaddbegin \DIFadd{negligible }\DIFaddend test-set correlation (R = \DIFdelbegin \DIFdel{-0.041),
indicating that generalizable models were not identified}\DIFdelend \DIFaddbegin \DIFadd{0.005)}\DIFaddend .
Feature importance for models of the left accumbens area were dominated
by three species of \emph{Bacteroides}, \emph{B. vulgatus} (3.3\% importance),
\emph{B. ovatus} (\DIFdelbegin \DIFdel{3.7}\DIFdelend \DIFaddbegin \DIFadd{3.2}\DIFaddend \% importance), and
\emph{B. uniformis} (\DIFdelbegin \DIFdel{2.9}\DIFdelend \DIFaddbegin \DIFadd{2.8}\DIFaddend \% importance).

While RF models for multiple brain regions had many important microbial taxa,
others were dominated by a small number of taxa, many of which
were also identified in LMs and RF models of cognitive function (Figures 2 and 3). 
In general, for all segments, a consistent number of species between \DIFdelbegin \DIFdel{31 and 45 }\DIFdelend \DIFaddbegin \DIFadd{28 and 42 }\DIFaddend (median = \DIFdelbegin \DIFdel{37}\DIFdelend \DIFaddbegin \DIFadd{38}\DIFaddend , less than 30\% of the input features)
was responsible for half of the cumulative importance,
regardless of model performance (Figure \DIFdelbegin \DIFdel{4A-B}\DIFdelend \DIFaddbegin \DIFadd{4C-E}\DIFaddend , Table S2).
We observed two major patterns of importance distribution from
species over the brain segments; some species portrayed high contributions
to multiple different segments, while others contributed modestly to
just one or two brain segments. Notable cases of the first pattern
included \DIFdelbegin \DIFdel{seven }\DIFdelend \DIFaddbegin \DIFadd{nine }\DIFaddend species - 
\emph{Anaerostipes hadrus},
\emph{B. vulgatus},
\emph{\DIFdelbegin \DIFdel{Fusicatenibacter saccharivorans}\DIFdelend \DIFaddbegin \DIFadd{B. ovatus}\DIFaddend },
\DIFaddbegin \emph{\DIFadd{F. saccharivorans}}\DIFadd{,
}\DIFaddend \emph{Ruminococcus torques},
\emph{Eubacterium rectale},
\emph{Coprococcus comes}\DIFaddbegin \DIFadd{,
}\emph{\DIFadd{A. celatus}} \DIFaddend and
\emph{B. wexlerae}, which combined, account for approximately 10\% of the cumulative relative importances,
computed after fitness-weighting on the reported brain segments (Figure \DIFdelbegin \DIFdel{4A-B}\DIFdelend \DIFaddbegin \DIFadd{4D-E}\DIFaddend ).

\begin{sidewaysfigure}[!htb]
    \centering
    \DIFdelbegin %DIFDELCMD < \includegraphics[width=\textwidth]{assets/Figure4.png}
%DIFDELCMD <     %%%
\DIFdelend \DIFaddbegin \small
    \includegraphics[width=0.9\textwidth]{assets/Figure4.png}
    \DIFaddend \caption{
        Microbial feature importance in predicting \DIFaddbegin \DIFadd{MSEL subscores and }\DIFaddend brain volumes in children over 18 months of age.
        (\textbf{A}) \DIFaddbegin \DIFadd{Test set correlations for RF models predicting MSEL subscale scores,
        with or without demographics included as predictors.
        (}\textbf{\DIFadd{B}}\DIFadd{) Important taxa in models predicting MSEL subscale scores.
        (}\textbf{\DIFadd{C}}\DIFadd{) }\DIFaddend Average test-set correlations for prediction of MRI segmentation
        data from microbiome and demographics on select segments after repeated
        cross validation. (\DIFdelbegin \textbf{\DIFdel{B}}%DIFAUXCMD
\DIFdelend \DIFaddbegin \textbf{\DIFadd{D}}\DIFaddend ) Heatmap of average individual relative taxa
        importances on each brain segment. Importances are reported as
        proportions relative to the sum of importances for each model - since
        every model is trained with 132 features, and even distribution of
        importance would be 0.75\% for each feature. Segments and taxa ordered
        by HCA on a select list of species with high ranks on average importances,
        and their respective highest-load segments (\DIFdelbegin \textbf{\DIFdel{C}}%DIFAUXCMD
\DIFdelend \DIFaddbegin \textbf{\DIFadd{E}}\DIFaddend ).
    }
    \label{fig:4}
\end{sidewaysfigure}

Among these, \emph{A. hadrus} is the variable with 
highest average relative importance along all reported \DIFaddbegin \DIFadd{models
}\DIFaddend (mean = \DIFdelbegin \DIFdel{1.7}\DIFdelend \DIFaddbegin \DIFadd{1.4}\DIFaddend \%; min = \DIFdelbegin \DIFdel{1.0}\DIFdelend \DIFaddbegin \DIFadd{0.8}\DIFaddend \%; max = \DIFdelbegin \DIFdel{3.8}\DIFdelend \DIFaddbegin \DIFadd{3.6}\DIFaddend \%, Figure \DIFdelbegin \DIFdel{4B-C}\DIFdelend \DIFaddbegin \DIFadd{4D-E}\DIFaddend ).
\emph{B. wexlerae}, which was important in RF models of cognitive function, 
\DIFdelbegin \DIFdel{and was the only species to be significantly associated
with cogntive function in young children using LMs, 
was }\DIFdelend \DIFaddbegin \DIFadd{was }\DIFaddend also important in models of the entorhinal, fusiform and lingual areas
and had the highest importance in models predicting the size of
both the left and right right parahippocampus.
Another cluster of model importance contained taxa associated with the
basal forebrain, especially the cingulate and the accumbens. This
cluster contains \emph{B. ovatus} and \emph{B. uniformis},
but also \emph{Alistipes finegoldii and \DIFdelbegin \DIFdel{Streptococcus salivaris}\DIFdelend \DIFaddbegin \DIFadd{S. salivarius}\DIFaddend }.

\DIFdelbegin \DIFdel{Both }\emph{\DIFdel{A. equolofaciens}} %DIFAUXCMD
\DIFdel{and }\DIFdelend \emph{A. celatus}, \DIFdelbegin \DIFdel{two closely-related
}\DIFdelend \DIFaddbegin \DIFadd{an }\DIFaddend equol-producing species, \DIFdelbegin \DIFdel{are }\DIFdelend \DIFaddbegin \DIFadd{is an }\DIFaddend examples of the second contribution
pattern, where a taxon only contributes significantly to a very small number of segments. 
\DIFdelbegin \DIFdel{Each of these species were }\DIFdelend \DIFaddbegin \DIFadd{This species was }\DIFaddend highly important in predicting the relative volume of
the right anterior cingulate (\DIFdelbegin \DIFdel{respectively, 3.0\% and 2.6\% importances}\DIFdelend \DIFaddbegin \DIFadd{2.8\% relative importance}\DIFaddend ).
Another example of this pattern is \emph{\DIFdelbegin \DIFdel{R. gnavus}%DIFDELCMD < \MBLOCKRIGHTBRACE%%%
\DIFdel{, which was the only
species with significant negative association with cognitive function in linear models.
It }\DIFdelend \DIFaddbegin \DIFadd{Coprococcus eutactus}}
\DIFadd{which }\DIFaddend is heavily associated with the left \DIFdelbegin \DIFdel{pars opercularis (2.7}\DIFdelend \DIFaddbegin \DIFadd{precentral area (3.5}\DIFaddend \% importance). 
Finally, \emph{C. comes} has an importance
distribution that splits almost evenly among the two previously reported
clusters. Its loading on the prediction of the left posterior cingulate
is the highest for a microbe in RF models (\DIFdelbegin \DIFdel{4.0}\DIFdelend \DIFaddbegin \DIFadd{3.7}\DIFaddend \% importance);
only age had higher relative importance in any model. At the same time,
\textit{C. comes} is one of the most important predictors for neighboring areas of the
posterior cingulate such as the pars opercularis (importances,
Left = \DIFdelbegin \DIFdel{2.3}\DIFdelend \DIFaddbegin \DIFadd{1.9}\DIFaddend \%, Right = \DIFdelbegin \DIFdel{2.8}\DIFdelend \DIFaddbegin \DIFadd{2.5}\DIFaddend \%), along with upper regions like the left
precentral (\DIFdelbegin \DIFdel{2.3}\DIFdelend \DIFaddbegin \DIFadd{2.0}\DIFaddend \% importance) and paracentral lobes (importances, Left = \DIFdelbegin \DIFdel{2.6}\DIFdelend \DIFaddbegin \DIFadd{2.0}\DIFaddend \%, Right = \DIFdelbegin \DIFdel{1.9}\DIFdelend \DIFaddbegin \DIFadd{1.5}\DIFaddend \%).

\section*{Discussion}

The relationship between the gut microbiome and brain function via the
gut-microbiome-brain axis has gained increasing acceptance largely as a
result of human epidemiological studies investigating atypical
neurocognition (eg anxiety and depression, neurodegeneration, attention
deficit / hyperactivity disorder, and autism \cite{romanGutBrainAxis2018,fosterGutBrainAxis2013})
and mechanistic studies in animal models
\cite{hsiaoMicrobiotaModulateBehavioral2013,needhamGutderivedMetaboliteAlters2022}.
The results from these studies point to the possibility
that gut microbes and their metabolism may be causally implicated in
cognitive development, but this study is the first to our knowledge that
directly investigates microbial species and their genes in relation to
typical development in young, healthy children. Understanding the gut-brain-microbiome axis
in early life is particularly important, since differences or
interventions in early life can have outsized and longer-term
consequences than those at later ages
due to the dynamic and plastic nature of both the gut microbiome and the brain.
Further, even in the absence of
causal impacts of microbial metabolism, identifying risk factors that
could point to other early interventions would also have value.

We identified several species in the family Eggerthelaceae that were
associated with cognitive function, including \emph{\DIFdelbegin \DIFdel{G.pamelaeae, }\DIFdelend A. \DIFdelbegin \DIFdel{equolofaciens, A. }\DIFdelend celatus}
(\DIFdelbegin \DIFdel{formerly regarded }\DIFdelend a subspecies of \emph{A. equolofaciens}
\cite{takahashiCompleteGenomeSequence2021}),
and \emph{Eggerthella lenta}. Many members of this family are
known in part due to unique metabolic activities. For example, \emph{A.
equolofaciens} produces the nonsteroidal estrogen equol from isoflaven
found in soybeans \cite{wangEnantioselectiveSynthesisSEquol2005}.
This is particularly intriguing, since \DIFdelbegin \DIFdel{both }\emph{\DIFdel{A. equolofaciens}} %DIFAUXCMD
\DIFdel{and }\DIFdelend \emph{A. celatus}
\DIFdelbegin \DIFdel{were }\DIFdelend \DIFaddbegin \DIFadd{was }\DIFaddend also the most important features in RF models of the
right anterior cingulate, which has been linked to social cognition
and reward-based decision making \cite{appsAnteriorCingulateGyrus2016,boesRightAnteriorCingulate2008,bushDorsalAnteriorCingulate2002},
and estrogen has previously been shown to have activity 
in this brain region \cite{xiaoEstrogenAnteriorCingulate2013}.
\DIFaddbegin \DIFadd{Another Eggerthelaceae, }\DIFaddend \emph{\DIFdelbegin \DIFdel{G. }\DIFdelend \DIFaddbegin \DIFadd{Gordonibacter }\DIFaddend pamelaeae}\DIFaddbegin \DIFadd{,
}\DIFaddend also has a unique metabolism that may impact the brain,
as it can metabolize the polyphenol ellagic acid
(found in pomegranates and some berries) into urolithin, which has been
shown in some studies to have a neuroprotective effect
\cite{gongUrolithinAlleviatesBloodbrain2022,selmaDescriptionUrolithinProduction2014}.
\DIFaddbegin \DIFadd{Though }\textit{\DIFadd{G. pamelaeae}} \DIFadd{was not identified
as an important feature in RF models,
it was also relatively low prevalence in our dataset,
occuring in only ~5\% of samples at greater than 2\% abundance
(in contrast to }\textit{\DIFadd{A. celatus}}\DIFadd{, which was in 32\% of samples).
}\DIFaddend While we could not find published evidence of neurological roles for \emph{E. lenta},
it has been extensively studied for
its ability to metabolize drug compounds such as the plant-derived heart
medication digoxin \cite{haiserPredictingManipulatingCardiac2013}.
The metabolic versatility of this clade and the large number of
species that are associated with cognition make these microbes prime
targets for further mechanistic studies.

\DIFaddbegin \DIFadd{We did not identify any statistical associations between individual species
and cognitive performance in our youngest subset (0 to 6 months).
We hypothesize that this may be because brain developmental trajectories
begin }\textit{\DIFadd{in utero}}\DIFadd{, and microbial influences (that can only begin at birth)
take additional time to manifest.
In line with this hypothesis, RF models that were able to learn associations
between early-life microbiomes and future cognitive function scores (Figure 3).
The top-ranked species for these models included several 
taxa that were important for predicting concurrent cognitive assessment scores after 18 months, notably
pathobionts in the family }\textit{\DIFadd{Enterobacteriaceae}} \DIFadd{such as
}\textit{\DIFadd{Escherichia}} \DIFadd{and }\textit{\DIFadd{Klebsiella}}\DIFadd{. 
Representatives of these genera are known to be early and long-term persistent
colonizers of infant intestine that can, in some settings,
promote inflammatory response \mbox{%DIFAUXCMD
\cite{popeEnterobacEarlyGut2019}}\hskip0pt%DIFAUXCMD
.
By contrast, several inflammation-reducing representatives
of the }\textit{\DIFadd{Bifidobacterium}} \DIFadd{genus,
from the early HMO-metabolizer }\textit{\DIFadd{B. longum}} \DIFadd{to the
later complex carbohydrate relying }\textit{\DIFadd{B. pseudocatenulatum}} \DIFadd{\mbox{%DIFAUXCMD
\cite{chungBifidoPsudocatenulatum2021}}\hskip0pt%DIFAUXCMD
,
commonly associated with weaning to solid food in early age,
also showed high relative importance.
The involvement of most important taxa in different aspects of inflammatory response
resonate with previous findings that associate chronic inflammation with
changes to neural responses in some types of stimulae \mbox{%DIFAUXCMD
\cite{xieChronicInfNeuro2019}}\hskip0pt%DIFAUXCMD
.
}

\DIFaddend Other bacteria that were associated with both cognitive ability and
brain structure had also have possible connections to previous studies.
For example, RF models predicting the size of the left nucleus accumbens
were dominated by \textit{Bacteroides} species,
especially \textit{B. vulgatus}, \textit{B. uniformis}, and \textit{B. ovatus}.
\textit{B. ovatus} has been shown to produce
a number of neuroactive compounds including SCFAs and neurotransmitters
such as GABA \cite{horvathBacteroidesOvatusColonization2022}.
Another \textit{Bacteroides} species, \textit{B. fragilis} was
able to ameliorate ASD-like symptoms in a mouse model
by consuming a microbially-dependent metabolite 4-ethylphenylsulfate (4EPS)
\cite{hsiaoMicrobiotaModulateBehavioral2013},
while \textit{B. ovatus} is a primary producer of a precursor to 4EPS
\cite{needhamGutderivedMetaboliteAlters2022} and increased anxiety
behaviors in mice.
The fact that the importance of these species in RF models,
and the high performance of RF models in predicting this region
were unilateral (affecting the left but not the right accumbens area)
was also notable. Most healthy individuals have a
rightward asymmetry in the nucleus accumbens,
which is involved in reward learning and risk taking behaviours
\cite{ernstAmygdalaNucleusAccumbens2005,yauNucleusAccumbensResponse2012}.
Reduced asymmetry between right and left nucleus accumbens
has been associated with substance use disorder in young adults
\cite{caoMappingCorticalSubcortical2021}. The accumbens area is also
associated with reward control, and in individuals diagnosed with ADHD,
it has been shown to have a divergent neuromorphology
\cite{hoogmanSubcorticalBrainVolume2017}.

\DIFaddbegin \DIFadd{Our ability to probe individual cognitive subscales provided
results that also align with previous findings and provide insight to
potential gut-microbial connections between cognition and neuroanatomy.
For example, a longitudinal neurocognitive development study pointed at bilateral
associations between the thalamus and the expressive language
component of the Mullen scales \mbox{%DIFAUXCMD
\cite{chenLongitudinalBrainMullen2023}}\hskip0pt%DIFAUXCMD
.
In line with this finding, we found that taxa such as 
}\textit{\DIFadd{B. wexlerae}} \DIFadd{and }\textit{\DIFadd{A. hadrus}} \DIFadd{to be among the top 15\% important predictors
on both left and right thalamus and the expressive language subscale.
Another example of those findings is a previously reported association between
the central opercular cortex (COC) and gross motor development \mbox{%DIFAUXCMD
\cite{jiangBrainNetworkCognition2023}}\hskip0pt%DIFAUXCMD
.
In this case, we identified that
butyrate-producing }\textit{\DIFadd{F. saccharivorans}} \DIFadd{is among the 15\% most important predictors
for both the gross motor subscale and three bilateral components of the COC
(left and right pars orbitalis, pars opercularis and pars triangularis).
Previous results also link to relationships between the caudate and the cerebellum,
and the visual reception score \mbox{%DIFAUXCMD
\cite{deoniMyelinationMullenPreprint2022}}\hskip0pt%DIFAUXCMD
.
}\textit{\DIFadd{S. salivarius}} \DIFadd{was consistently found among the top 20\% predictors for this subscale, along with
both sides of the thalamus and cerebellum white matter, along with cerebellar vermal lobules I-X.
}

\DIFaddend As is made apparent by the potentially opposing roles of 
\textit{B. fragilis} and \textit{B. ovatus},
individual species from the same genus may play different roles.
As such, the use of shotgun metagenomic sequencing
may be crucial to identifying microbial influences on the brain,
since it enables species-level resolution of microbial taxa.
A previous study of
cognition in 3 year old subjects used 16S rRNA gene amplicon sequencing
and showed that genera from the Lachnospiraceae family as well as
unclassified Clostridiales (now Eubacteriales) were associated with
higher scores on the Ages and Stages Questionnaire
\cite{sordilloAssociationInfantGut2019}.
However, each of these clades encompass dozens of genera with
diverse functions, each of which may have different effects. Indeed,
several of the taxa that were positively associated with cognitive
function in this study, including \emph{B. wexlerae}, \emph{D.
longicatena}, \emph{R. faecis}, and \emph{A. finegoldii} are
Clostridiales, as is \emph{R. gnavus}, which we found was negatively
associated with cognitive function (Figure 2A-B). This kind of
species-level resolution is typically not possible with amplicon
sequencing.

In addition to improved species-level resolution, shotgun metagenomic
sequencing also enables gene-functional insight. We showed that
genes for the metabolism of SCFAs, both their degradation and synthesis,
are associated with cognitive function scores (Figure 2). 
SCFAs are produced by anaerobic fermentation of dietary fiber
and have been linked with immune system regulation as well as directly
with brain function \cite{dalileRoleShortchainFatty2019}.
Microbial metabolism of other potentially neuroactive molecules
were also associated with cognitive ability. For example, we were
particularly interested in genes for metabolizing glutamate, which is a
critical neurotransmitter controlling neuronal excitatory/inhibitory
signaling along with gamma-aminobutyric acid (GABA), and their balance
in the brain controls neural plasticity and learning, particularly in
the developing brain \cite{cohenkadoshLinkingGABAGlutamate2015,palomo-buitragoGlutamateInteractionsObesity2019}.
However, while the
differential abundance of genes for the metabolism of neuroactive
compounds like these is suggestive, it is difficult to reason about the
relationship between levels of these genes and the gut concentrations of
the molecules their product enzymes act on since the presence or absence
of metabolites may select for microbes with the ability to degrade
or produce that metabolite respectively.
For this reason, future studies coupling shotgun
metagenomics with stool metabolomics could improve our understanding of
the relationship between microbial metabolism and cognitive development.
\DIFaddbegin \DIFadd{Serum metabolomic analysis may be even more informative,
since the metabolites measured in stool are not necessarily indicative
of exposure, but drawing sufficient blood from children,
especially very young children, is challenging.
}\DIFaddend Further, strain-level analysis linking specific gene content in species
of interest could further refine targeted efforts at identifying
specific metabolic signatures of microbe-brain interactions.

The use of multiple age-appropriate cognitive assessments that could be
normalized to a common scale enabled us to analyze microbial
associations across multiple developmental periods, but carries several
drawbacks. In particular, the test-retest reliability, as well as
systematic differences between test administrators may introduce
substantial noise into these observations, particularly in the youngest
children. In addition, our study period overlapped with the beginning of
the COVID-19 pandemic, and we and others have observed some reduction in
measured scores for children that were assessed after the implementation
of lockdowns. In our subject set for this study, these effects are more
pronounced in some age groups due to the period when active recruitment for
the study was occurring
\cite{blackwellYouthWellbeingCOVID192022,deoniImpactCOVID19Pandemic2021}
(Figure S3).

This analysis allowed us to establish links between microbial taxa and
their functional potential with cognition and brain structure. 
Although we \DIFdelbegin \DIFdel{cannot test causality or the chemistry behind the interactions
}\DIFdelend \DIFaddbegin \DIFadd{did not directly test the causal relationships
}\DIFaddend between gut microbial taxa and their genes, the gut, and the brain,
this study provides clear and statistically significant associations
\DIFdelbegin \DIFdel{between the infant and early child gut microbiota and neurocognition}\DIFdelend \DIFaddbegin \DIFadd{that could serve as targets for future efforts in pre-clinical models.
For example, mouse models of brain development, learning, and executive function
could be assessed in germ-free animals with the addition of
metabolically characterized and manipulated microbes}\DIFaddend .
Future studies should \DIFaddbegin \DIFadd{also }\DIFaddend focus on characterizing the early-life microbiome
and neurocognitive development across different geographic regions
and lifestyles such as covering traditionally understudied populations,
such as low-resource urban, peri-urban and rural communities
to obtain the more comprehensive understanding of the
variability within the different gut microbiomes reflects on
neurocognition. These studies would also provide us with the wealth of
data on different strains from the same species to better understand the
effect of genes and their products. Furthermore, culturing and microbial
community enrichment studies combined with genetic manipulation and
genomic approaches to understand microbial metabolism at the molecular
level is key, as the metabolic functions shape and influence the
human host and its health. The discovery of the neuroactive metabolites
could provide us with biomarkers for early detection or necessary
medicinally useful molecules that can be applied in intervention.

\section*{Material and methods}

\subsubsection*{Study Ethics}

All procedures for this study were approved by the local institutional
review board at Rhode Island Hospital, 
\DIFdelbegin \DIFdel{and }\DIFdelend \DIFaddbegin \DIFadd{(Investigating Early Childhood Neurodevelopment 1500991,
and Prenatal Influences on Fetal and Infant Brain Development 1501119;
most recent approval for both: 2020-01-15)
and }\DIFaddend all experiments adhered to
the regulation of the review board. Written informed consent was
obtained from all parents or legal guardians of enrolled participants.

\subsubsection*{Participants}

Data used in this study were drawn from an ongoing longitudinal
brain and cognitive development, termed RESONANCE, based in Providence, RI, USA,
and spanning fetal through adolescent development. The
RESONANCE cohort is part of the NIH Environmental influences
on Child Health Outcomes (ECHO) study
\cite{forrestAdvancingScienceChildren2018,gillmanEnvironmentalInfluencesChild2018},
and includes neuroimaging (magnetic
resonance imaging, MRI), neurocognitive assessments, bio-specimen
analyses, subject genetics, broad environmental exposure data
(such as chemical, water quality, nutrition, etc.), and
rich demographic, socioeconomic, family and medical history information.

As a study of neurotypical development, mothers and their children
\DIFdelbegin \DIFdel{were devoid of }\DIFdelend \DIFaddbegin \DIFadd{exclusion criteria included }\DIFaddend major risk factors for abnormal or atypical development\DIFdelbegin \DIFdel{.
These included }\DIFdelend \DIFaddbegin \DIFadd{,
i.e., }\DIFaddend preterm birth (\textless 37 weeks gestation) low birth weight (\textless 1500g),
\DIFdelbegin \DIFdel{in utero }\DIFdelend \DIFaddbegin \textit{\DIFadd{in utero}} \DIFaddend exposure to alcohol, cigarette or illicit substance exposure;
abnormalities on fetal ultrasound; non-singleton or complicated pregnancy,
including preeclampsia, high blood pressure, or gestational diabetes;
complicated vaginal or cesarian birth; 5 minute APGAR scores \textless 8;
NICU admission; neurological disorder in the child
(e.g., head injury resulting in loss of consciousness, epilepsy);
and psychiatric or learning disorder (including maternal depression)
in the infant, parents, or siblings requiring medication in the year prior to pregnancy.
In addition to screening at
the time of enrollment, on-going screening for worrisome behaviors using
validated tools was performed to identify at-risk children and remove
them from subsequent analysis.
For this study, \DIFdelbegin \DIFdel{361 }\DIFdelend \DIFaddbegin \DIFadd{381 }\DIFaddend typically-developing children between the
ages of \DIFdelbegin \DIFdel{2.8 months }\DIFdelend \DIFaddbegin \DIFadd{40 days }\DIFaddend and 10 years \DIFdelbegin \DIFdel{old }\DIFdelend (median age 2 years, 2 months) were
selected for analysis based on having provided at least one stool sample
in the same week as being evaluated using one of the cogntitive ability
assessments (MSEL, WPPSI, WISC-V).

\subsubsection*{Cognitive Assessments}

General cognitive ability was assessed using age-appropriate performance-based measures.
For children under 3 years of age, we used the Early Learning
Composite (ELC) from the Mullen Scales of Early Learning (MSEL)
\cite{mullenMullenScalesEarly1995}.
The MSEL isa standardized and population-normed tool that assesses 
function in 5 major domains: fine and gross motor, expressive and receptive language,  
and visual functioning.
The ELC is an age-normalized composite derived from these individual domains
with a mean of 100 and standard deviation of 15. For older children, full-scale IQ (FSIQ)
was assessed using the Wechsler Preschool and Primary Scale of Intelligence 4th Edition
(WPPSI-IV, children 3 to 6) \cite{wechslerWechslerPreschoolPrimary2012},
and the Wechsler Intelligence Scale for Children 5th Edition (WISC-V)
for children older than 6 \cite{wechslerWechslerIntelligenceScale1949}. 
Like the ELC, FSIQ is normalized to a mean of 100 and standard deviation of 15.


\subsubsection*{Stool Sample Collection and Sequencing}

Stool samples (n = 493) were collected by parents in OMR-200 tubes
(OMNIgene GUT, DNA Genotek, Ottawa, Ontario, Canada), stored on ice, and
brought within 24 hrs to the lab in RI where they were immediately
frozen at -80$^{\circ}$C. Stool samples were not collected if the subject had
taken antibiotics within the last two weeks. DNA extraction was
performed at Wellesley College (Wellesley, MA). Nucleic acids were
extracted from stool samples using the RNeasy PowerMicrobiome kit,
excluding the DNA degradation steps. Briefly, the samples were lysed by
bead beating using the Powerlyzer 24 Homogenizer (Qiagen, Germantown,
MD) at 2500 rpm for 45 s and then transferred to the QIAcube (Qiagen,
Germantown, MD) to complete the extraction protocol. Extracted DNA was
sequenced at the Integrated Microbiome Resource (IMR, Dalhousie
University, NS, Canada).

Shotgun metagenomic sequencing was performed on all samples. A pooled
library (max 96 samples per run) was prepared using the Illumina Nextera
Flex Kit for MiSeq and NextSeq from 1 ng of each sample. Samples were
then pooled onto a plate and sequenced on the Illumina NextSeq 550
platform using 150+150 bp paired-end "high output";' chemistry,
generating 400 million raw reads and 120 Gb of sequence per plate.

\subsubsection*{Computational analysis of metagenomes}

Shotgun metagenomic sequences were analyzed using the bioBakery suite of
computational tools \cite{beghiniIntegratingTaxonomicFunctional2021}.
First, KneadData (v0\DIFdelbegin \DIFdel{.7.7}\DIFdelend \DIFaddbegin \DIFadd{.10.0}\DIFaddend ) was used to perform quality
control of raw sequence reads, such as read trimming and removal of
reads matching a human genome reference. Next, MetaPhlAn (v3\DIFaddbegin \DIFadd{.1}\DIFaddend .0\DIFdelbegin \DIFdel{.7}\DIFdelend , using
database mpa\_\DIFdelbegin \DIFdel{v30}\DIFdelend \DIFaddbegin \DIFadd{v31}\DIFaddend \_CHOCOPhlAn\_201901) was used to generate taxonomic
profiles by aligning reads to a reference database of marker genes.
Finally, HUMAnN (v3\DIFaddbegin \DIFadd{.6}\DIFaddend .0\DIFdelbegin \DIFdel{.0a4}\DIFdelend ) was used to functionally profile the
metagenomes.

\subsubsection*{Machine learning for cognitive development}

Prediction of cognitive scores was carried out as a set of regression
experiments targeting real-valued continuous assessment scores.
Different experiment sets were designed to probe how different
representations of the gut microbiome (taxonomic profiles, functional
profiles encoded as ECs) would behave, with and without the addition of
demographics (sex and maternal education as a proxy of socioeconomic
status) on participants from different age groups\DIFaddbegin \DIFadd{.
}

\DIFadd{For the concurrent cognitive assessment score preditions,
all viable datapoints containing 
age at data collection, demographics, stool metagenomics and cognitive assessment were employed.
The cognitive assessments were collected as either composite scores
(for the experiments in Figure 3) or, when available, subscores on the
Mullen Scales of Early Learning (Figure 4 A-B: expressive language, gross motor and visual reception).
For the concurrent prediction experiments, when multiple viable longitudinal samples were present,
the single earliest viable datapoint inside the age range of the experiment was considered.
For the future prediction experiments, every demographics-complete subject
contributed a single datapoint built from the combination of
the latest metagenome collected within the stool collection time range and
the latest future cognitive assessment on the assesment time range}\DIFaddend .
Age (in months) was provided as a covariate for all models (Table S3).

Random Forests (RFs)
\cite{breimanRandomForests2001}
were selected as the prediction engine and processed using the
DecisionTree.jl \cite{sadeghiDecisionTreeJlJulia2022}
implementation, inside the MLJ.jl \cite{blaomMLJJuliaPackage2020}.
Independent RFs were trained for each
experiment, using a set of default regression hyperparameters from
Breiman and Cutler \cite{breimanRandomForests2001},
on a repeated cross-validation approach with different RNG seeds. One hundred
repetitions of 3-fold CV with 10 different intra-fold RNG states each
were employed, for a total of 3000 experiments per input set.

After the training procedures, the root-mean-square error (RMSE) for
cognitive assessment scores and mean absolute proportional error (MAPE)
for the brain segmentation data, along with Pearson's
correlation coefficient (R) were benchmarked on the validation and train
sets. MAPE was chosen as the metric for brain segments due to magnitude
differences between median volumes of each segment, which would hinder
interpretation of raw error values without additional reference.

To derive biological insight from the models, the covariate variable
importances for all the input features, measured by mean decrease in
impurity (MDI, or GINI importance), was also analyzed. Leveraging the
distribution of results from the extensive repeated cross validation
experiments, rather than electing a representative model or picking the
highest validation-set correlation, a measure of model fitness
(\textbf{Equation 1}) was designed to weight the importances from each
trained forest, and all "importance" values reported refer to 
the average fitness-weighted importance accross all models.
The objective was to give more weight to those with
higher benchmarks on the validation sets (or higher generalizability),
while penalizing information from highly overfit models, drawing
inspiration from the approach used on another work employing repeated CV
on Random Forests with high-dimensional, low sample size
datasets \cite{woodruffInflammationAutoreactivityDefine2022}.
The resulting fitness-weighted importances were used
to generate the values in Figures 3-4.

\begin{equation}
    fitness = \sqrt{max(r_{train}, 0) \cdot max(r_{test}, 0)}
\end{equation}


\subsubsection*{MRI / segmentation}

T1-weighted anatomical MRI data were acquired using a 3-Dimensional MP-RAGE sequence
on 3T Siemens Trio scanner with the following parameters:
TE=5.6msec, TR=1400msec, TI = 950msec, FA=15 degrees, and 1.1x1.1x1.1mm resolution.
Image matrix and field-of-view were adjusted by infant size to achieve the desired image resolution.
For children under 4 years of age, MRI was performed during natural non-sedated sleep using previously described methods \cite{deanPediatricNeuroimagingUsing2014}.
Older children were scanned awake whilst watching a movie or favorite TV show.
To help minimize motion, pediatric (MedVac) immobilizers were used along with foam cushions and padding.
Image quality was monitored during and following scanning.
If motion artifacts were observed (e.g., ghosting, blurring, or other signal variations),
scans were repeated or removed from analysis.

Following data acquisition, brain region volumes corresponding to total white and gray matter,
cortical gray matter, brainstem, and left and right hemisphere lateral ventricles,
thalamus, caudate, putamen, pallidum, hippocampus, amygdala,
and nucleus accumbens were calculated using an atlas matching approach \cite{bruchhageLongitudinalBrainCognitive}.
Individual images were first nonlinearly aligned to MNI-space using
a multi-step registration approach (ANTs 2.2 toolbox \cite{avantsInsightToolKitImage2014}).
The MNI-aligned Harvard-Oxford brain atlas was then aligned and superimposed onto the individual child
by applying the inverse transformation, and the brain regions of interest delinated (segmented)
and their volume calculated by summing the number of image voxels within each \cite{jenkinsonFSL2012}.
Brain region volumes used in RF models were normalized to total brain volume
(the sum of white and gray matter).

\subsubsection*{Data and code availability}

All data needed to evaluate the conclusions in the paper
are present in the paper and/or the Supplementary Materials.
Taxonomic and functional microbial profiles, as well as subject
demographics necessary for statistical analyses and machine learning are
available on the Open Science Framework
\cite{bonhamECHORESONANCEMicrobiome2022}.
Data processing, generation of summary statistics, and
generation of plots was performed using the julia programming language
and 3rd party packages
\cite{bezansonJuliaFreshApproach2017,
    bonhamMicrobiomeJlBiobakeryUtils2021,
    danischMakieJlFlexible2021,
    blaomMLJJuliaPackage2020,
    ben_sadeghi_2022_7359268,
    dahua_lin_2023_7695673,
    douglas_bates_2023_7734970}.
Information for replicating the package environment
as well as code for data analysis and figure generation, as well as scripts for automated
download of input files are available on GitHub \cite{kevin_bonham_2023_7647510}.

\section*{References}

\printbibliography[heading=none]

\section*{Acknowledgements}
% https://www.science.org/content/page/science-advances-information-authors#acknowledgments

This research was funded by NIH UG3 OD023313 and Wellcome LEAP 1kD.
The authors declare that they have no competing interests.
We thank Libusha Kelly (Albert Einstein School of Medicine) and Daphne Koinis Mitchell (Brown University)
for helpful feedback on the manuscript. The research team also thanks the participants and families
of the participants who generously shared their time and samples for this project.

% https://credit.niso.org/

KSB, VKC, CH, and SD were responsible for conceptualization,
methodology and project administration.
VKC, SD, and VD were responsible for funding aqusition.
KSB, SHM, MB, FB, RCL, and JB contributed to data curation.
Formal analyses were conducted by KSB, GFB, and MB.
Investigation was performed by KSB, GFB, SHM, EW, FB, RCL, and MB.
Resources were provided by SHM, EW, FB, RCL, JB, CH, and MB.
Software and data visualizations were produced by KSB and GFB.
KSB, MB, SD, VD, and VKC were responsible for supervision.
Validation was performed by KSB, SHM, JB, and MB.
KSB and GFB wrote the initial draft, 
and review and editing were provided by SD, CH, and VKC.

\end{document}
